<!DOCTYPE html>
<html lang="en">
<head>
    <script src="http://www.google.com/jsapi" type="text/javascript"></script>

    <script type="text/javascript">google.load("jquery", "1.3.2");</script>

    <meta charset="UTF-8">
    <title>Results</title>
    <link rel="stylesheet" type="text/css" href="css/template.css">

    <!-- BOOTSTRAP -->
    <!-- Latest compiled and minified CSS -->
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">

    <!-- jQuery library -->
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>

    <!-- Latest compiled JavaScript -->
    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>

    <!-- a script that does the underlining for the side bar -->
    <script>

        $(document).ready(function(){

            $("#button-barc-hidden").click(function(){
                $("#hidden-barc").slideToggle("slow");
            });

            $("#button-organ-hidden").click(function(){
                $("#hidden-organ").slideToggle("slow");
            });

            $("#button-bulb-hidden").click(function(){
                $("#hidden-bulb").slideToggle("slow");
            });

            $("#barc-app1").click(function(){
                $("#applications-description").text("Considering the increasing frequency of natural disasters, having a cheap and immediate way to generate structures could be used to temporarily shelter people, seal leaks, or bridge over flooded areas. Microorganisms can be also engineered to simultaneously detect and/or feed off hazardous waste and help clean up septic systems.");
            });

            $("#barc-app2").click(function(){
                $("#applications-description").text("Barchitecture could also facilitate space exploration as currently significant trade-offs are made for the amount of necessary materials and equipment to be transported in outer space due to monetary and fuel considerations. Microbial biomining of silicates and metal oxides from Lunar regolith is an ongoing research area that aims to facilitate the manufacturing of circuits and spaceship components on the Moon. Barchitecture could facilitate this process by using bacteria to not only harvest silicates and produce and secrete biosilica but also form desired structures and bind biosilica to grow semiconductors and circuit boards.");
            });

            $("#barc-app3").click(function(){
                $("#applications-description").text("The production of small scale devices with medical applications also derives from our ability to grow structures from bacteria. Biopolymers, such as PHB, have a wide range of applications (agriculture films, safe materials for toys and packaging, feedstock additives). Via accurate light control, PHB-based medical implants can be produced, such as biocompatible veins and bronchus. See here the outcomes of our conversation with Blue Alpha - a company producing PHB from bacteria.");
            });


        });
    </script>
</head>

<style>

    .image-content-here {
        margin:20% 0 2% 0;
        height:70%;
        width:70%;
    }

    .image-content-hidden {
        margin:2% 0 2% 15%;
        height:60%;
        width:60%;
    }

    .summary-images {
        margin:80% 0 2% 0;
        height:90%;
        width:90%;
    }

    .overlay-buttons-here {
        position: relative;
        margin:1% 0 0 25% !important;
        background-color: transparent;
        border:none;
    }

    #description-intro {
        position:relative;
        font-size: 30px;
        text-align: center;
        font-family: "Josefin Sans";
        padding: 5% 0 0 15%;
        margin:0 !important;
    }

    .summary-here {
        position:relative;
        font-size: 15px;
        text-align: justify;
        font-family: Lato;
        padding: 15% 0 0 10%;
        margin:0 !important;
    }

    .overlay-text {
        position:relative;
        max-width:80%;
        font-size: 1.6vw;
        text-align: center;
        font-family: "Josefin Sans";
        padding: 2% 10% 5% 0;
        margin:0 !important;
    }

    .infos-here {
        font-size: 30px;
        color: black;
    }

    .infos-hidden-here {
        font-size: 3vw;
        color: black;
    }

    .side-images {
        height:75px;
        width:75px;
        margin:5% 0 5% 7%;
        border-radius: 50%;
        box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
    }

    .list-results {
        list-style-type: none;
        position:relative;
        max-width:90%;
        padding: 2% 3% 2% 2%;
        margin:0 !important;
    }

    .list-item-results {
        font-family: Lato;
        font-size: 1.3vw;
        text-align: justify;
        padding:3% 3% 2% 3%;
    }

    .applications-text {
        position:relative;
        font-size: 1.4vw;
        text-align: center;
        font-family: Lato;
        padding: 15% 12% 2% 10%;
        margin:0 !important;
    }

    #applications-description {
        position:relative;
        max-width:97%;
        font-size: 1.5vw;
        text-align: justify;
        font-family: Lato;
        padding: 15% 12% 21% 10%;
        margin:0 !important;
    }

    .buttons-here {
        border:none;
        margin:0 0 5% 35%;
        background-color: transparent;
    }

    .application-boxes {
        border-left:solid 1px rgba(39, 32, 34, 0.16);
    }

   #hidden-barc, #hidden-organ, #hidden-bulb {
        display:none;
        margin-left:7.5%;
    }
</style>

<body data-spy="scroll" data-target="#side-index">


<!-- The top navbar -->
<div style="margin:0 !important; padding:0 !important" class="container-fluid">
    <div id="top" class="row">

        <img src="assets/logotop.png" class="logo-top">

        <ul class="navy">
            <li id="drop1" class="navy-link"><a href="#">PROJECT</a>
                <ul class="navy-hidden">
                    <li><a href="#" class="dropped">Description</a></li>
                    <li><a href="#" class="dropped">Results</a></li>
                    <li><a href="#" class="dropped">Parts</a></li>
                    <li><a href="#" class="dropped">Gold Criteria</a></li>
                    <li><a href="#" class="dropped">Interlab</a></li>

                </ul>
            </li>

            <li id="drop" class="navy-link"><a href="#">HUMAN PRACTICES</a>
                <ul class="navy-hidden">
                    <li><a href="#" class="dropped">Overview</a></li>
                    <li><a href="#" class="dropped">Public Engagement</a></li>
                    <li><a href="#" class="dropped">Integrated Practices</a></li>
                </ul>
            </li>

            <li id="drop5" class="navy-link"><a href="#">MODELLING</a>
                <ul class="navy-hidden">
                    <li><a href="#" class="dropped">Overview</a></li>
                    <li><a href="#" class="dropped">Barchitecture</a></li>
                    <li><a href="#" class="dropped">Game of LIT</a></li>
                    <li><a href="#" class="dropped">Mammalian</a></li>
                    <li><a href="#" class="dropped">Light Bulb</a></li>
                    <li><a href="#" class="dropped">Glossary</a></li>
                </ul>
            </li>

            <li id="drop3" class="navy-link"><a href="#">ENTREPRENEURSHIP</a>
                <ul class="navy-hidden">
                    <li><a href="#" class="dropped">Overview</a></li>
                    <li><a href="#" class="dropped">Product</a></li>
                    <li><a href="#" class="dropped">IP</a></li>
                    <li><a href="#" class="dropped">Business Environment</a></li>
                    <li><a href="#" class="dropped">Marketing</a></li>
                    <li><a href="#" class="dropped">Management</a></li>
                    <li><a href="#" class="dropped">Economic Appraisal</a></li>
                </ul>
            </li>


            <li id="drop2" class="navy-link"><a href="#about">MEDALS</a>
                <ul class="navy-hidden">
                    <li><a href="#" class="dropped">Bronze</a></li>
                    <li><a href="#" class="dropped">Silver</a></li>
                    <li><a href="#" class="dropped">Gold</a></li>
                </ul>
            </li>

            <li id="drop6" class="navy-link"><a href="#about">PEOPLE</a>
                <ul class="navy-hidden">
                    <li><a href="#" class="dropped">Team</a></li>
                    <li><a href="#" class="dropped">Collaborations</a></li>
                    <li><a href="#" class="dropped">Attributions</a></li>
                </ul></li>

            <li id="drop4" class="navy-link"><a href="#">NOTE BOOK</a>
                <ul class="navy-hidden">
                    <li><a href="#" class="dropped">Protocols</a></li>
                    <li><a href="#" class="dropped">Safety</a></li>
                    <li><a href="#" class="dropped">Stuff</a></li>
                </ul>
            </li>

        </ul>

    </div>

</div>


<!-- Content-->
<div class="container-fluid">
    <div class="row">

        <!-- The side navbar -->
        <nav id="side-index" class="col-sm-2">
            <ul class="nav nav-stacked" data-spy="affix" data-offset-top="205">
                <li id="step2" class="navy-link side-link"><a href="#barc"><img src="assets/icon-barc.png" class="side-images"><p>Barchitecture</p></a></li>
                <li id="step3" class="navy-link side-link"><a href="#organ"><img src="assets/icon-organ.png" class="side-images"><p>Organ printing</p></a></li>
                <li id="step4" class="navy-link side-link"><a href="#bulb"><img src="assets/icon-bulb.png" class="side-images"><p>Bacterial bulb</p></a></li>
            </ul>
        </nav>

        <!-- Content-->
        <div style=" margin-left: 4%" class="col-sm-9">

            <!-- Overview -->
            <div class="row">

                <div class="col-sm-10">
                    <p id="description-intro">Results from our 3 applications that we have developed for optogenetics. All organic.</p>
                </div>

                <!-- Barchitecture -->
                <div class="col-sm-4">
                    <img src="assets/icon-barc.png" class="image-content-here">

                    <p class="overlay-text">Applying light switches to create 3D structures</p>

                </div>

                <!-- Organ Printing-->
                <div class="col-sm-4">
                    <img src="assets/icon-organ.png" class="image-content-here">
                    <p class="overlay-text">Using light to precisely control organ printing</p>

                </div>

                <!-- Light bulb-->
                <div class="col-sm-4">
                    <img src="assets/icon-bulb.png" class="image-content-here">
                    <p class="overlay-text">A bacterial light-bulb controlled through light</p>

                </div>

            </div>

            <!-- Barchitecture-->
            <div id="barc" class="row">

                <div style="border-top: solid 1px rgba(39, 32, 34, 0.16);margin-bottom:3%" class="col-sm-5">
                    <p class="summary-here"><b>Barchitecture</b> is a 3D printing technology based on 2 suspended colonies of E. Coli cells that display on their surface one of the 2 binding partners that form <b>a strong covalent bond</b>: SpyTag or SpyCatcher. By using light in the form of lasers, one ensures that only in the desired location in the cell population covalent bonds will be formed between bacteria. Once a 3D structure is formed, <b>light induces the production of PHB</b>, which accumulates in the form of granules inside cells. Cells then use a type-2 system to <b>secrete the PHB</b>, with the granules being cross-linked into a solid shape by a photoinitiator.</p>
                </div>

                <div style="border-top: solid 1px rgba(39, 32, 34, 0.16);margin-bottom:3%" class="col-sm-2">

                    <img src="assets/icon-barc.png" class="summary-images">

                    <button id="button-barc-hidden" type="button" class="btn btn-default btn-sm overlay-buttons overlay-buttons-here">
                        <span class="glyphicon glyphicon-chevron-down infos-hidden-here"></span>
                    </button>
                </div>

                <div style="border-top: solid 1px rgba(39, 32, 34, 0.16);margin-bottom:3%" class="col-sm-5">

                    <h1 class="title">Barchitecture Results</h1>
                    <ul class="list-results">
                        <li style="background-color: rgba(59,57,69,0.2)" class="list-item-results">Designed <b>4 biobricks</b> and experiments for 2 systems to induce <b>cell adhesion</b> to form 3D upon <b>light exposure</b></li>
                        <li style="background-color: rgba(120,117,133,0.2)" class="list-item-results">Performed experiments to <b>test blue-light activation</b> for the transcription of necessary proteins</li>
                        <li style="background-color: rgba(59,57,69,0.2)" class="list-item-results"><b>Modelled</b> the expression of adhesion proteins on E. Coli cell surface</li>
                        <li style="background-color: rgba(120,117,133,0.2)" class="list-item-results"><b>Modelled</b> cell adhesion with <b>a cellular automaton</b> for better visualisation</li>
                    </ul>
                </div>

                <div id="hidden-barc" class="col-sm-10">

                    <!-- Applications of barchitecture-->
                    <div class="row">

                        <h1 style="text-align: center; padding:3% 0 5% 0 !important;" class="title">Why Barchitecture is useful</h1>

                        <div style="background-color: rgba(32,88,137,0.12); border-top:solid 1px rgba(39, 32, 34, 0.16)" class="col-sm-3 application-boxes">

                            <p class="applications-text">Natural disasters</p>

                            <button id="barc-app1" type="button" class="btn btn-default btn-sm buttons-here">
                                <span class="glyphicon glyphicon-chevron-right infos-here"></span>
                            </button>
                        </div>

                        <div style="float:right; border:solid 1px rgba(39, 32, 34, 0.16); height:397px; overflow: scroll !important;" class="col-sm-9">
                            <p id="applications-description">Considering the increasing frequency of natural disasters, having a cheap and immediate way to generate structures could be used to temporarily shelter people, seal leaks, or bridge over flooded areas. Microorganisms can be also engineered to simultaneously detect and/or feed off hazardous waste and help clean up septic systems.</p>
                        </div>

                        <div style="background-color: rgba(137,2,19,0.12)" class="col-sm-3 application-boxes">

                            <p class="applications-text">Space Exploration</p>

                            <button id="barc-app2" type="button" class="btn btn-default btn-sm buttons-here">
                                <span class="glyphicon glyphicon-chevron-right infos-here"></span>
                            </button>
                        </div>

                        <div style="background-color: rgba(15,137,0,0.12); border-bottom:solid 1px rgba(39, 32, 34, 0.16)" class="col-sm-3 application-boxes">
                            <p class="applications-text">Medical implants</p>

                            <button id="barc-app3" type="button" class="btn btn-default btn-sm buttons-here">
                                <span class="glyphicon glyphicon-chevron-right infos-here"></span>
                            </button>

                        </div>
                    </div>

                    <!-- Why use cells to do it-->

                    <!-- Scientific process-->
                    <div class="row">

                        <!-- Overview - here should add buttons to navigate to each bit (or put on the side)-->
                        <div class="col-sm-11">
                            <p class="text-content-new">We developed (i) transcriptional and (ii) post-translational light switches for the induction of cellular aggregation. The final engineered cells would be able to respond to the light stimuli and assemble into desired structures that can simultaneously produce and bind a desired polymer.</p>
                        </div>

                        <!-- Cell Adhesion-->
                        <div class="text-section col-sm-11">
                            <h1 class="title">1. Transcriptional light switch for cell adhesion</h1>

                            <p class="text-content-new">Using the blue light inducible promoter PBlind , we aim to use blue light (6hr, 475nm) to induce the transcription of Intimin, a cell surface protein, fused to either SpyTag (13 amino acids) or SpyCatcher (138 amino acids, 15kDa). SpyTag and SpyCatcher are two binding partners that come from CnaB2 (immunoglobulin-llike collagen adhesin domain) of the FbaB protein, found in the invasive strains of S. pyogenes. SpyTag contains a reactive aspartate that forms a strong isopeptide bond with a reactive lysine residue in SpyCatcher when in close proximity. </p>

                            <!-- the 2 briobricks-->
                            <img src="assets/BB-Intimin-SpyTag.png" class="image-content-hidden">

                            <img src="assets/BB-Intimin-SpyCatcher.png" class="image-content-hidden">

                            <p class="text-content-new">Once a mixture of cells are light induced to display SpyTag or SpyCatcher on their cell surface, they are expected to aggregate. Similar to lithography, structures would be generated layer by layer from a flat surface by exposing a particular radius with blue light. </p>

                            <p class="text-content-new">Designers note: after analysing multiple cell surface display proteins, we opted for Intimin (EaeA intimin) as it has shown to display passenger proteins of similar size to SpyCatcher and it had not been characterised by iGEM teams before. Other cell candidates considered include: ice nucleation protein, PgsA and OmpC.</p>

                            <img src="assets/results-di-spy.png" class="image-content-hidden">

                            <img src="assets/results-di-photoactiv.png" class="image-content-hidden">

                            <p class="text-content-new">Designed Experiments</p>

                            <p class="text-content-new">A) Represents two cell lines expressing either intimin’-SpyTag or intimin’-SpyCatcher fusion proteins. Cells harbouring either construct under the control of a constitutive promoter would have either been mixed together or mixed with WT cells (control) (15min) or not mixed and passed through a particle sizer with a range of aperture sizes (0.4µm - 40µm) to compare aggregate sizes. As secondary experiments and for visual purposes we would have also incubated cells expressing Intimin-SpyTag with purified GFP-SpyCatcher, Intimin’-SpyCatcher with purified GFP-SpyTag and Intimin-GFP-SpyTag with Intimin-SpyCatcher cell lines for 15min and visualize cell-protein binding and cell-cell interaction via fluorescence microscopy. We also included a His-Tag within all SpyCatcher construct variants to enable in-vitro analysis of complexes formed with SpyTag variants.</p>

                            <p class="text-content-new">B) Represents two cell lines expressing either intimin’-SpyTag or intimin’-SpyCatcher fusion proteins pre and post light induction. Aggregation experiments described in A) would have been repeated after replacing the constitutive promoter with PBlind , blue-light inducible promoter and transforming cells with EL222 (blue light inducer) under a constitutive promoter (C). Cell aggregation and fluorescence experiments would have been carried out with cells grown overnight under darkness or under blue-light and in the presence or absence of the EL222 construct.</p>

                            <img src="assets/exp-barc-1.jpg" class="image-content-hidden">

                            <img src="assets/exp-barc-2.jpg" class="image-content-hidden">

                            <img src="assets/exp-barc-3.jpg" class="image-content-hidden">

                            <img src="assets/exp-barc-4.jpg" class="image-content-hidden">

                            <img src="assets/GFP-SpyTag.jpg" class="image-content-hidden">
                        </div>

                        <!-- Light Induced Control-->
                        <div class="text-section col-sm-11">
                            <h1 class="title">2. Post-translational light switch for cell adhesion</h1>

                            <p class="text-content-new">As a more immediate induction of cell aggregation, we aimed to develop a post-translational light switch in which cells are constitutively expressing either Intimin’ fusion proteins with SpyTag or an inactive version of SpyCatcher. Only upon light exposure, SpyCatcher is able to bind SpyTag resulting in a much faster response than transcriptional induction. This is achieved by incorporating a photocaged unnatural amino acid (UAA), Ne-methyl-L-lysine, in place of the reactive lysine in SpyCatcher required for the covalent bond formation with the SpyTag aspartate residue. Upon exposure to UV light (20min, 365nm), the “cage” group in the unnatural photocaged amino acid is cleaved off revealing the native amino acid and a biologically active protein. This approach also adds a layer of bio-containment as cells will only function when externally supplied with UAA.</p>

                            <p class="text-content-new">To achieve this, we introduced an amber stop codon (TAG) in place of the reactive Lys 31 residue (Lys31X) in SpyCatcher. Amberless E. coli cells also have to express pyrrolysyl tRNA (pyIT/tRNAPylCUA) from M. mazei and pyrrolysyl-tRNA synthetase from M. barkeri and the UAA must be supplemented in the media. The pyrrolysyl-tRNA synthetase catalyses the acylation of the suppressor tRNACUA with the UAA. During translation, the UAG amber codon in the mRNA is recognized by the acylated tRNACUA and the UAA will be added to the growing polypeptide chain. The orthogonality of this system has shown to work in both E. coli and mammalian cells.</p>

                            <img src="assets/results-di-photocage.png" class="image-content-hidden">

                            <img src="assets/BB-Intimin-SpyCatcher%20amber.png" class="image-content-hidden">

                            <img src="assets/results-di-photocage2.png" class="image-content-hidden">

                            <p class="text-content-new">Figure 2. Post-translational light induction of cell aggregation. A) Represents two cell lines expressing constitutively either intimin’-SpyTag or the photocaged version of intimin’-SpyCatcher fusion proteins. Cells harbouring either construct would have either been mixed together or mixed with WT cells (control) or not mixed and exposed to UV light (365nm) or not exposed for 25min-1hr and then passed through a particle sizer with a range of aperture sizes (0.4µm - 40µm) to compare aggregate sizes. As secondary experiments and for visual purposes we would have also incubated cells expressing Intimin’-SpyTag with purified photocaged GFP-SpyCatcher, photocaged Intimin’-SpyCatcher with purified GFP-SpyTag and Intimin-GFP-SpyTag with photocaged Intimin-SpyCatcher cell lines pre and post photo-lysis (365nm) for 25min-1hr to visualize cell-protein binding and cell-cell interaction via fluorescence microscopy. We also included a His-Tag within all photocaged SpyCatcher construct variants to enable in-vitro analysis of complexes formed with SpyTag variants.</p>

                        </div>

                        <!-- Polymer production/binding-->
                        <div class="text-section col-sm-11">
                            <h1 class="title">3. Polymer production/binding</h1>

                            <p class="text-content-new">Bacteria has been widely used for the production of biomaterials to generate sustainable and eco friendly bricks, bioplastic products, items of clothing as well as “living materials” by incorporating nanoparticles into biofilms. With Barchitecture we can light-induce cellular 3D structural arrangements while producing, binding or degrading biomaterials such as PHA or silicates.</p>

                            <p class="text-content-new">Proposed design for the production, secretion and binding of PHA granules:</p>

                            <img src="assets/results-di-phb.png" class="image-content-hidden">

                            <p class="text-content-new">Through a similar mechanism, bacteria, could be light-guided to form precise and intricate structures that can then simultaneously produce and bind biosilica. This approach will enable “growing” electronics just with the guidance of a particular wavelength of light.</p>

                            <p class="text-content-new">Our engineered cells would form the desired 3D structure using blue light. These cells would also express photocaged (inactive) recombinant silicatein on their surface. Another wavelength of light would activate silicatein to begin the production of biosilica from water-soluble biosilica precursors added to the media. Our cells could also present ligands on their surface to attach the produced polymer to their surface.</p>
                        </div>


                        <!-- Actual results-->
                        <div class="text-section col-sm-11">
                            <p class="title">Our wet lab data</p>

                            <p class="text-content-new">We tested whether the promoter has any significant leakage. Also, we wanted to show that GFP cannot be expressed in the absence of EL222. This is of particular interest as the aim of LIT is to demonstrate the versatility and high precision of light control. As shown in Figure1, only J23151-GFP (positive control) had a significant difference in fluorescence compared to R0040-GFP (negative control) WT cells and the Luria Broth (LB) in both dark and Blue-light conditions.  Pblind-GFP had no significantly different fluorescence level compared to the LB baseline, negative control or WT cells in either condition. This is expected, as the EL222 protein is required for blue-light inducible transcriptional activation.</p>

                            <img src="assets/results-graph-barc.png" class="image-content-hidden">

                            <p class="text-content-new">Figure 1. Blue light inducible promoter (Pblind) characterisation. J23151 is a constitutive promoter, R0040 is a TetR repressible promoter (repression inhibited only by the addition of tetracycline), Pblind promoter is a fusion of EL222 (photosensitive transcription factor) binding region and the luxI promoter, where EL222 is only able to dimerize and bind the Pblind promoter upon blue light exposure, where it can then recruit RNAP and drive the transcription of genes downstream. Wild type (WT) 10beta cells were transformed with J23151-GFP (positive control), R0040-GFP (negative control) or Pblind-GFP. All cells were grown overnight at 37°C in darkness or exposed to blue light (465nm) and diluted to OD600=0.6 to record GFP fluorescence. The data represents the mean of 3 biological replicates and 4 technical replicates for each condition. Luria broth (LB) was included as a baseline for the fluorescence. Error bars represent the SD and statistical significance of **** P < 0.0001 was calculated using the Tukey's multiple comparisons test.</p>
                        </div>

                    </div>

                </div>

            </div>

            <!-- Tissue printing -->
            <div id="organ" class="row">
                <div style="border-top: solid 1px rgba(39, 32, 34, 0.16);margin-bottom:3%" class="col-sm-5">
                    <p class="summary-here">Barchitecture is a 3D printing technology based on 2 suspended colonies of E. Coli cells that display on their surface one of the 2 binding partners that form a strong covalent bond: SpyTag or SpyCatcher. By using light in the form of lasers, one ensures that only in the desired location in the cell population covalent bonds will be formed between bacteria. Once a 3D structure is formed, light induces the production of PHB, which accumulates in the form of granules inside cells. Cells then use a type-2 system to secrete the PHB, with the granules being cross-linked into a solid shape by a photoinitiator.</p>
                </div>


                <div style="border-top: solid 1px rgba(39, 32, 34, 0.16);margin-bottom:3%" class="col-sm-2">

                    <img src="assets/icon-organ.png" class="summary-images">

                    <button id="button-organ-hidden" type="button" class="btn btn-default btn-sm overlay-buttons overlay-buttons-here">
                        <span class="glyphicon glyphicon-chevron-down infos-hidden-here"></span>
                    </button>

                </div>

                <div style="border-top: solid 1px rgba(39, 32, 34, 0.16);margin-bottom:3%" class="col-sm-5">

                    <h1 class="title">Organ printing Results</h1>
                    <ul class="list-results">
                        <li style="background-color: rgba(59,57,69,0.2)" class="list-item-results">Designed biobricks and experiments for 2 systems of how cells bind to each other to form 3D upon light exposure</li>
                        <li style="background-color: rgba(120,117,133,0.2)" class="list-item-results">Performed experiments to test blue-light activation for the transcription of necessary proteins</li>
                        <li style="background-color: rgba(59,57,69,0.2)" class="list-item-results">Modelled the expression of adhesion proteins on E. Coli cell surface</li>
                        <li style="background-color: rgba(120,117,133,0.2)" class="list-item-results">Modelled cell adhesion with a cellular automaton for better visualisation</li>
                    </ul>
                </div>

                <div id="hidden-organ" class="col-sm-10">

                </div>
            </div>

            <!-- Light bulb -->
            <div id="bulb" class="row">

                <div style="border-top: solid 1px rgba(39, 32, 34, 0.16);margin-bottom:3%" class="col-sm-5">
                    <p class="summary-here">Our bacterial bulb uses E. Coli to luminescence only during night time and cyanobacteria to provide nutrients for the bioluminescent bacteria. We incorporate the Lux operon into E. coli and suppress its activity with the EL222 photosensitive protein. We use an existent glucose secretion system for the cyanobacteria (reference).</p>
                </div>

                <div style="border-top: solid 1px rgba(39, 32, 34, 0.16); margin-bottom:3%" class="col-sm-2">

                    <img src="assets/icon-bulb.png" class="summary-images">

                    <button id="button-bulb-hidden" type="button" class="btn btn-default btn-sm overlay-buttons overlay-buttons-here">
                        <span class="glyphicon glyphicon-chevron-down infos-hidden-here"></span>
                    </button>
                </div>

                <div style="border-top: solid 1px rgba(39, 32, 34, 0.16); margin-bottom:3%" class="col-sm-5">

                    <h1 class="title">Bacterial bulb Results</h1>
                    <ul class="list-results">
                        <li style="background-color: rgba(59,57,69,0.2)" class="list-item-results">Designed the bulb with a light-induced transcriptional control of bioluminescence</li>
                        <li style="background-color: rgba(120,117,133,0.2)" class="list-item-results">Modelled the dimensions necessary for the bulb to accommodate cyanobacteria</li>
                        <li style="background-color: rgba(59,57,69,0.2)" class="list-item-results">3D printed a prototype for the bulb after integrating designs from artists</li>
                    </ul>
                </div>

                <div id="hidden-bulb" class="text-section col-sm-10">
                    <p class="text-content-new">The light sensitive E. coli detects sunlight (blue light) to repress bioluminescence encoded by LuxCDABE. In the presence of light, transcription of the LuxCDABE for the production of Luciferase and substrates will be inhibited by EL222 binding onto our blue light repressible promoter (PBLrep). In the dark, the inhibition will be released and bioluminescence will be induced. This ensures that our bulb is only active in the dark and excess buildup of substrates and luciferase, potentially toxic, will be prevented. This system also ensures that cell concentration dependent luminescence through quorum sensing is surpassed and our cultures can fluoresce independently and in response to light changes.</p>

                    <img src="assets/results-di-bulb.png" class="image-content-hidden">

                    <p class="text-content-new">The blue light repressible promoter (PBLrep) consists of the 18bp DNA binding region of EL222, a natural photosensitive DNA-binding protein from the marine bacterium Erythrobacter litoralis HTCC2594, positioned between the -35 and -10 regions of the RNAP binding site. In the dark, EL222 is inactive as its N-terminal LOV domain represses its DNA-binding C-terminal HTH domain. In the daytime, exposure to blue light (450nm) results in the LOV-HTH interaction to be released, allowing it to dimerize and bind its binding region, causing steric hindrance to RNAP binding, ultimately repressing transcription. Therefore, only at nighttime the transcription of LuxCDABE will occur.</p>

                    <p class="text-content-new">Cells would have been transformed with (construct) and (construct el222) (constitutively expressing EL222 protein) plasmids. Cells would have been either kept in the dark or exposed to blue light (465 nm) for 6 hours and bioluminescence levels would have been measured by aliquoting samples into a 96-well plate every 6 hours using FLUOstar plate reader.</p>

                    <img src="assets/bulb-proto.png" class="image-content-hidden">

                    <img src="assets/bulb-proto-2.png" class="image-content-hidden">

                    <p class="text-content-new">Through mathematical modelling, the optimal dimensions and structure of the bulb were determined (click here for modelling details). The circular structure of the LIT bulb maximizes the exposure of Cyanobacteria to sunlight. A pump was introduced to ensure the cells remain suspended and the cell culture is homogeneously distributed throughout the LIT bulb. A three-way manual valve was introduced to allow for the easy replacement of media once every 12 months. A filter will be added to the valve to allow media to be replaced without removing the cell culture.</p>

                    <p class="text-content-new">Biosafety and microbial containment considerations are of particular importance in the manufacturing design process of the LIT Bulb, as our product intends to coexist with the wider population as a source of public illumination.</p>

                    <p class="text-content-new">The bulb has been designed to ensure biocontainment and prevent the interaction with our engineered microorganisms or their release into the environment.</p>

                    <p class="text-content-new">The energy strain for the production of bioluminescence by our engineered E. coli cells is so large that they would be outcompeted quickly by other microorganisms if they were to be released in the environment.</p>

                    <p class="text-content-new">Cyanobacteria will be engineered to secrete the glucose product from photosynthesis, thus they would be unable to sustain their growth when competing against other microbes in the environment if they were to be released.</p>

                </div>

            </div>

        </div>

        <a href="#" id="myBtn" title="Go to top">Back to Top</a>

    </div>
</div>

</body>

</html>