<html lang="en">
<head>

    <link href='https://fonts.googleapis.com/css?family=Josefin+Sans' rel='stylesheet' type='text/css'>
    <link href='https://fonts.googleapis.com/css?family=Noto+Sans' rel='stylesheet' type='text/css'>
    <link href='https://fonts.googleapis.com/css?family=Lato' rel='stylesheet' type='text/css'>
    <link rel="stylesheet" type="text/css" href="css/template.css">

    <script src="http://www.google.com/jsapi" type="text/javascript"></script>

    <meta charset="UTF-8">
    <title>LIT Results</title>

    <!-- BOOTSTRAP -->
    <!-- Latest compiled and minified CSS -->
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">

    <!-- jQuery library -->
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>

    <!-- Latest compiled JavaScript -->
    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>


    <script>

        $(document).ready(function(){

            //the second menu buttons will have some initial values
            var section1 = $('#barc-nav1'), section2 = $('#barc-nav2'), section3 = $('#barc-nav3'), section4 = $('#barc-nav4');
            var section5 = $('#tissue-nav1'), section6 = $('#tissue-nav2'), section7 = $('#tissue-nav3'), section8 = $('#tissue-nav4');
            var section9 = $('#bulb-nav1'), section10 = $('#bulb-nav2'), section11 = $('#bulb-nav3'), section12 = $('#bulb-nav4');

            //these functions give values to the high-level buttons that define what section 1, 2, 3 and 4
            // and to the relevant content that should be displayed

            $("#barc").click(function(){
                $(section5).hide();
                $(section6).hide();
                $(section7).hide();
                $(section8).hide();
                $(section9).hide();
                $(section10).hide();
                $(section11).hide();
                $(section12).hide();

                $(section1).show();
                $(section2).show();
                $(section3).show();
                $(section4).show();


            });

            $("#tissue").click(function(){


                $(section1).hide();
                $(section2).hide();
                $(section3).hide();
                $(section4).hide();
                $(section12).hide();
                $(section9).hide();
                $(section10).hide();
                $(section11).hide();

                $(section5).show();
                $(section6).show();
                $(section7).show();
                $(section8).show();

            });

            $("#bulb").click(function(){


                $(section1).hide();
                $(section2).hide();
                $(section3).hide();
                $(section4).hide();
                $(section5).hide();
                $(section6).hide();
                $(section7).hide();
                $(section8).hide();

                $(section9).show();
                $(section10).show();
                $(section11).show();
                $(section12).show();

            });

            //the second menu buttons - when we press on the button, it should fadeIn the right content

            $(section1).click(function(){
                //here we only have to fade the right content and out the bad one
                $('#barc-1').fadeIn("slow");
                $('#barc-2').hide();
                $('#barc-3').hide();
                $('#barc-4').hide();

                $('#tissue-1').hide();
                $('#tissue-2').hide();
                $('#tissue-3').hide();
                $('#tissue-4').hide();

                $('#bulb-1').hide();
                $('#bulb-2').hide();
                $('#bulb-3').hide();
                $('#bulb-4').hide();


            });

            $(section2).click(function(){
                //here we only have to fade the right content and out the bad one
                $('#barc-2').fadeIn("slow");

                $('#barc-1').hide();
                $('#barc-3').hide();
                $('#barc-4').hide();

                $('#tissue-1').hide();
                $('#tissue-2').hide();
                $('#tissue-3').hide();
                $('#tissue-4').hide();

                $('#bulb-1').hide();
                $('#bulb-2').hide();
                $('#bulb-3').hide();
                $('#bulb-4').hide();
            });

            $(section3).click(function(){
                //here we only have to fade the right content and out the bad one
                $('#barc-3').fadeIn("slow");
                $('#barc-2').hide();
                $('#barc-1').hide();
                $('#barc-4').hide();

                $('#tissue-1').hide();
                $('#tissue-2').hide();
                $('#tissue-3').hide();
                $('#tissue-4').hide();

                $('#bulb-1').hide();
                $('#bulb-2').hide();
                $('#bulb-3').hide();
                $('#bulb-4').hide();
            });

            $(section4).click(function(){
                //here we only have to fade the right content and out the bad one
                $('#barc-4').fadeIn("slow");
                $('#barc-2').hide();
                $('#barc-3').hide();
                $('#barc-1').hide();

                $('#tissue-1').hide();
                $('#tissue-2').hide();
                $('#tissue-3').hide();
                $('#tissue-4').hide();

                $('#bulb-1').hide();
                $('#bulb-2').hide();
                $('#bulb-3').hide();
                $('#bulb-4').hide();
            });

            $(section5).click(function(){
                //here we only have to fade the right content and out the bad one
                $('#tissue-1').fadeIn("slow");
                $('#tissue-2').hide();
                $('#tissue-3').hide();
                $('#tissue-4').hide();

                $('#barc-1').hide();
                $('#barc-2').hide();
                $('#barc-3').hide();
                $('#barc-4').hide();

                $('#bulb-1').hide();
                $('#bulb-2').hide();
                $('#bulb-3').hide();
                $('#bulb-4').hide();


            });

            $(section6).click(function(){
                //here we only have to fade the right content and out the bad one
                $('#tissue-2').fadeIn("slow");
                $('#tissue-1').hide();
                $('#tissue-3').hide();
                $('#tissue-4').hide();

                $('#barc-1').hide();
                $('#barc-2').hide();
                $('#barc-3').hide();
                $('#barc-4').hide();

                $('#bulb-1').hide();
                $('#bulb-2').hide();
                $('#bulb-3').hide();
                $('#bulb-4').hide();
            });

            $(section7).click(function(){
                //here we only have to fade the right content and out the bad one
                $('#tissue-3').fadeIn("slow");
                $('#tissue-2').hide();
                $('#tissue-1').hide();
                $('#tissue-4').hide();

                $('#barc-1').hide();
                $('#barc-2').hide();
                $('#barc-3').hide();
                $('#barc-4').hide();

                $('#bulb-1').hide();
                $('#bulb-2').hide();
                $('#bulb-3').hide();
                $('#bulb-4').hide();
            });

            $(section8).click(function(){
                //here we only have to fade the right content and out the bad one
                $('#tissue-4').fadeIn("slow");
                $('#tissue-2').hide();
                $('#tissue-3').hide();
                $('#tissue-1').hide();

                $('#barc-1').hide();
                $('#barc-2').hide();
                $('#barc-3').hide();
                $('#barc-4').hide();

                $('#bulb-1').hide();
                $('#bulb-2').hide();
                $('#bulb-3').hide();
                $('#bulb-4').hide();
            });

            $(section9).click(function(){

                //here we only have to fade the right content and out the bad one
                $('#bulb-1').fadeIn("slow");

                $('#bulb-2').hide();
                $('#bulb-3').hide();
                $('#bulb-4').hide();

                $('#barc-1').hide();
                $('#barc-2').hide();
                $('#barc-3').hide();
                $('#barc-4').hide();

                $('#tissue-1').hide();
                $('#tissue-2').hide();
                $('#tissue-3').hide();
                $('#tissue-4').hide();

            });

            $(section10).click(function(){
                //here we only have to fade the right content and out the bad one
                $('#bulb-2').fadeIn("slow");

                $('#bulb-1').hide();
                $('#bulb-3').hide();
                $('#bulb-4').hide();

                $('#barc-1').hide();
                $('#barc-2').hide();
                $('#barc-3').hide();
                $('#barc-4').hide();

                $('#tissue-1').hide();
                $('#tissue-2').hide();
                $('#tissue-3').hide();
                $('#tissue-4').hide();
            });

            $(section11).click(function(){
                //here we only have to fade the right content and out the bad one
                $('#bulb-3').fadeIn("slow");

                $('#bulb-2').hide();
                $('#bulb-1').hide();
                $('#bulb-4').hide();

                $('#barc-1').hide();
                $('#barc-2').hide();
                $('#barc-3').hide();
                $('#barc-4').hide();

                $('#tissue-1').hide();
                $('#tissue-2').hide();
                $('#tissue-3').hide();
                $('#tissue-4').hide();
            });

            $(section12).click(function(){
                //here we only have to fade the right content and out the bad one
                $('#bulb-4').fadeIn("slow");

                $('#bulb-2').hide();
                $('#bulb-3').hide();
                $('#bulb-1').hide();

                $('#barc-1').hide();
                $('#barc-2').hide();
                $('#barc-3').hide();
                $('#barc-4').hide();

                $('#tissue-1').hide();
                $('#tissue-2').hide();
                $('#tissue-3').hide();
                $('#tissue-4').hide();
            });



            //functions for the small tables inside the sectionContent variables
            $("#barc-app1").click(function(){
                $("#applications-description").text("Considering the increasing frequency of natural disasters, having a cheap and immediate way to generate structures could be used to temporarily shelter people, seal leaks, or bridge over flooded areas. Microorganisms can be also engineered to simultaneously detect and/or feed off hazardous waste and help clean up septic systems. To this end, biological light switches would prove useful in having a way of precisely guiding the phenotypes desired from the bacteria.");
            });

            $("#barc-app2").click(function(){
                $("#applications-description").text("From our conversation with the European Space Agency, we found out about the significant trade-offs made for the amount of necessary materials and equipment to be transported in outer space due to monetary and fuel considerations. Barchitecture could then facilitate space exploration by only having to bring engineered cells into space and make them resistant to the harsh environments. We also learned about the microbial biomining of silicates and metal oxides from Lunar regolith, which aims to facilitate the manufacturing of circuits and spaceship components on the Moon. Barchitecture could facilitate this process by using bacteria to not only harvest silicates and produce and secrete biosilica, but also form desired structures and bind biosilica to grow semiconductors and circuit boards.");
            });

            $("#barc-app3").click(function(){
                $("#applications-description").text("We also talked to Blue Alpha, a company producing PHB from bacteria. Different to the needs expressed by architects (e.g. PHB structures to grow and then modify themselves), we learnt from them about the production of small scale devices with medical applications.  According to Blue Alpha’s clients, PHB has a wide range of applications (agriculture films, safe materials for toys and packaging, feedstock additives). The why behind Barchitecture is then motivated by the potential to transfer the technology from large architectural structures to PHB-based medical implants, such as biocompatible veins and bronchus.");
            });

            $("#tissue-app1").click(function(){
                $("#applications-tissue-description").text("Scaffold design and construction methods need to be tailored towards the tissue tried to be produced. Scaffolds for heart reconstruction need to fulfil completely different prerequisites to bone scaffolds. With the incredibly diverse microenvironment of cells a myriad of different designs have been tested. Furthermore, the rapid invention of new biomaterials that are usable as scaffold material increase the number of possible scaffolds even further.");
            });

            $("#tissue-app2").click(function(){
                $("#applications-tissue-description").text("After cells are seeded onto the scaffold their growth is limited by the constraints posed by the scaffold. This puts further responsibility on scaffold design and reduces the flexibility in the design. Once seeded, cells grow into the scaffold and reliable prediction of cell organisation is extremely difficult in a growing 3-dimensional tissue. Limited only by the constraints of the scaffold cells can form separate aggregations inside the scaffold or certain types of cell can establish a dominance instead of forming a coexistence with their neighbouring cells.");
            });

            $("#tissue-app3").click(function(){
                $("#applications-tissue-description").text("Controlling this differentiation process has recently been achieved by embedding the growing tissue into a flow chamber and exposing the cells to different growth factors that guide the differentiation. [3] This proves to be very efficient but it also means that all cells in the tissue are exposed to the same growth factors and targeted differentiation of specific cells is difficult.");
            });

            $("#tissue-app4").click(function(){
                $("#applications-tissue-description2").text("In order to form structures cells need to form dynamic non-covalent formations in adherens junctions that allow constant remodelling of the structure while still maintaining a high degree of rigidity. The cell adhesion proteins in these junctions are stabilized via connections with the cytoskeleton, most notably actin filaments. Cells forming synthetic tissues therefore need to have the ability to form these connections rapidly in a cell culture that is dividing.In mammalian cells transcription takes around 30 min, translation (incl. mRNA export) another 30 min and further protein sorting and processing is dependent on the the type of protein. For the purpose of forming structural patterns, a photosensitive cell adhesion system therefore needs to be post-translational and instantaneous.");
                $("#hidden-list").hide();
            });

            $("#tissue-app5").click(function(){
                $("#applications-tissue-description2").text("Many studies in developmental biology focus on the cell signalling in a growing organisms to elucidate the formation of complex structures. From these studies we know that simple adhesion between cells is only the first step of forming a stable, 3-dimensional structure. Equally important is the activation of genes that lead to the production of the extracellular matrix (ECM) and other cell adhesion proteins like integrins and selections.Furthermore, the ability to activate specific genes in a cell, theoretically, allows biological engineers to harness the already existing code to activate a full range of mechanisms.");
                $("#hidden-list").show();
            });

        });
    </script>
</head>

<style>

    .image-content-here {
        margin:15% 0 2% 15%;
        height:40%;
        width:40%;
    }

    .image-content-hidden {
        margin:2% 0 2% 15%;
        height:60%;
        width:60%;
    }

    .summary-images {
        margin:80% 0 2% 0;
        height:90%;
        width:90%;
    }

    .overlay-buttons-here {
        position: relative;
        margin:1% 0 0 5% !important;
        background-color: transparent;
        border:none;
    }

    #description-intro {
        position:relative;
        font-size: 1.6vw !important;
        text-align: center !important;
        font-family: "Josefin Sans" !important;
        padding: 3% 5% 0 15%;
        margin:0 !important;
    }

    .summary-here {
        position:relative;
        font-size: 1.5vw !important;
        text-align: justify;
        font-family: Lato !important;
        padding: 5% 15% 0 10% !important;
        margin:0 !important;
    }

    .overlay-text {
        position:relative;
        max-width:80%;
        font-size: 1.6vw !important;
        text-align: center !important;
        font-family: "Josefin Sans" !important;
        padding: 2% 10% 5% 0;
        margin:0 !important;
    }

    #hidden-list {
        display: none;
        margin: 0 7% 0 7% !important;
        list-style: none !important;
    }
    .infos-here {
        font-size: 30px;
        color: black;
    }

    .infos-hidden-here {
        font-size: 3vw;
        color: black;
    }

    .side-images {
        height:75px;
        width:75px;
        margin:5% 0 5% 7%;
        border-radius: 50%;
        box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
    }

    .list-results {
        list-style-type: none;
        position:relative;
        max-width:90%;
        padding: 2% 3% 2% 2%;
        margin:0 !important;
    }

    .list-item-results {
        font-family: Lato;
        font-size: 1.3vw;
        text-align: justify;
        padding:3% 3% 2% 3%;
    }

    .applications-text {
        position:relative;
        font-size: 1.4vw !important;
        text-align: center !important;
        font-family: Lato !important;
        padding: 15% 5% 2% 5%;
        margin:0 !important;
    }

    #applications-description, #applications-tissue-description, #applications-tissue-description2 {
        position:relative;
        max-width:97%;
        font-size: 1.5vw !important;
        text-align: justify !important;
        font-family: Lato !important;
        padding: 5% 10% 5% 10%;
        margin:0 !important;
    }

    .buttons-here {
        border:none;
        margin:0 0 5% 5% !important;
        background-color: transparent;
    }

    .application-boxes {
        border-left:solid 1px rgba(39, 32, 34, 0.16);
        margin: 3% 0 0 0;
    }

    #hidden-barc, #hidden-organ, #hidden-bulb {
        display:none;
        margin-left:7.5%;
    }

    #barc:hover {
        opacity:0.5;
    }

    #tissue:hover {
        opacity:0.5;
    }

    #bulb:hover {
        opacity:0.5;
    }

    #barc, #tissue, #bulb  {
        animation: float 2.7s ease-in-out infinite;
    }

    @keyframes float {
        0% {

            transform: translatey(0px);
        }
        50% {

            transform: translatey(-6px);
        }
        100% {

            transform: translatey(0px);
        }
    }

    #barc-1, #barc-2, #barc-3, #barc-4, #tissue-1, #tissue-2, #tissue-3, #tissue-4, #bulb-1, #bulb-2, #bulb-3, #bulb-4 {
        display:none;
    }

    #barc-nav1, #barc-nav2, #barc-nav3, #barc-nav4, #tissue-nav1, #tissue-nav2, #tissue-nav3, #tissue-nav4, #bulb-nav1, #bulb-nav2, #bulb-nav3, #bulb-nav4 {
        display:none;
        width:20% !important;
        height:70px;
        margin:1% 2% 2% 1% !important;
        background-color: rgba(0, 67, 147, 0.42);
    }

    .buttoni {
        position:relative;
        font-size: 1.3vw !important;
        font-family: Lato !important;
        text-align: center !important;
        padding: 0 0 0 0 !important;
        margin:0 !important;
        color: white;
    }


</style>

<body data-spy="scroll" data-target="#side-index">


<!-- The top navbar -->
    <div style="margin:0 !important; padding:0 !important" class="container-fluid">
    <div id="top" class="row">

        <a href="http://2017.igem.org/Team:UCL"><img src="http://2017.igem.org/wiki/images/9/92/Logotop.png" class="logo-top"></a>

        <ul class="navy">
            <li id="drop1" class="navy-link"><a href="http://2017.igem.org/Team:UCL/Description">PROJECT</a>
                <ul class="navy-hidden">
                    <li><a href="http://2017.igem.org/Team:UCL/Description" class="dropped">Description</a></li>
                    <li><a href="http://2017.igem.org/Team:UCL/Results" class="dropped">Results</a></li>
                    <li><a href="http://2017.igem.org/Team:UCL/Parts" class="dropped">Parts</a></li>
                    <li><a href="http://2017.igem.org/Team:UCL/Part_Improvement" class="dropped">Gold Part</a></li>
                    <li><a href="http://2017.igem.org/Team:UCL/InterLab" class="dropped">Interlab</a></li>

                </ul>
            </li>

            <li id="drop" class="navy-link"><a href="http://2017.igem.org/Team:UCL/Human_Practices">HUMAN PRACTICES</a>
                <ul class="navy-hidden">
                    <li><a href="http://2017.igem.org/Team:UCL/Human_Practices" class="dropped">Overview</a></li>
                    <li><a href="http://2017.igem.org/Team:UCL/Engagement" class="dropped">Public Engagement</a></li>
                    <li><a href="http://2017.igem.org/Team:UCL/HP/Gold_Integrated" class="dropped">Integrated Practices</a></li>
                </ul>
            </li>

            <li id="drop5" class="navy-link"><a href="#">MODELLING</a>
                <ul class="navy-hidden">
                    <li><a href="http://2017.igem.org/Team:UCL/Model" class="dropped">Overview</a></li>
                    <li><a href="http://2017.igem.org/Team:UCL/Model4" class="dropped">Barchitecture</a></li>
                    <li><a href="http://2017.igem.org/Team:UCL/Model1" class="dropped">Game of LIT</a></li>
                    <li><a href="http://2017.igem.org/Team:UCL/Model3" class="dropped">Mammalian</a></li>
                    <li><a href="http://2017.igem.org/Team:UCL/Model2" class="dropped">Light Bulb</a></li>
                    <li><a href="#" class="dropped">Glossary</a></li>
                </ul>
            </li>

            <li id="drop3" class="navy-link"><a href="http://2017.igem.org/Team:UCL/Entrepreneurship">ENTREPRENEURSHIP</a>
                <ul class="navy-hidden">
                    <li><a href="http://2017.igem.org/Team:UCL/Entrepreneurship" class="dropped">Overview</a></li>
                    <li><a href="#" class="dropped">Product</a></li>
                    <li><a href="#" class="dropped">IP</a></li>
                    <li><a href="#" class="dropped">Business Environment</a></li>
                    <li><a href="#" class="dropped">Marketing</a></li>
                    <li><a href="#" class="dropped">Management</a></li>
                    <li><a href="#" class="dropped">Economic Appraisal</a></li>
                </ul>
            </li>


            <li id="drop2" class="navy-link"><a href="http://2017.igem.org/Team:UCL/Medals">OUR MEDALS</a></li>

            <li id="drop6" class="navy-link"><a href="#about">PEOPLE</a>
                <ul class="navy-hidden">
                    <li><a href="http://2017.igem.org/Team:UCL/Team" class="dropped">Team</a></li>
                    <li><a href="http://2017.igem.org/Team:UCL/Collaborations" class="dropped">Collaborations</a></li>
                    <li><a href="http://2017.igem.org/Team:UCL/Attributions" class="dropped">Attributions</a></li>
                </ul></li>

            <li id="drop4" class="navy-link"><a href="http://2017.igem.org/Team:UCL/Notebook">LAB BOOK</a></li>

        </ul>

    </div>

</div>


    <!-- Content-->
    <div class="container-fluid">
        <div class="row">

            <!-- Content-->
            <div style=" margin-left: 10% !important" class="col-sm-11">

                <!-- Overview -->
                <div class="row">

                    <div class="col-sm-10">
                        <p id="description-intro">Click on one of our applications to find our more about how we explored biological light switches</p>
                    </div>

                    <!-- Barchitecture -->
                    <div  class="col-sm-4">

                        <img id="barc" src="http://2017.igem.org/wiki/images/2/23/Icon-barc.png" class="image-content-here">

                        <p class="overlay-text">Applying light switches to create 3D structures</p>

                    </div>

                    <!-- Organ Printing-->
                    <div class="col-sm-4">
                        <img id="tissue" src="http://2017.igem.org/wiki/images/b/bb/Icon-organ.png" class="image-content-here">
                        <p class="overlay-text">Using light to precisely control tissue engineering</p>

                    </div>

                    <!-- Light bulb-->
                    <div class="col-sm-4">
                        <img id="bulb" src="http://2017.igem.org/wiki/images/8/89/Icon-bulb.png" class="image-content-here">
                        <p class="overlay-text">A bacterial light-bulb controlled through light</p>

                    </div>

                </div>

                <!-- The second menu the sections will be replaced when clicking on the images above, to be specific to each module-->
                <div class="row">
                    <div class="col-sm-11">
                        <!-- Menu for barchitecture-->
                        <button style=" background-color: rgb(96, 26, 210)" id="barc-nav1" type="button" class="btn btn-default btn-lg overlay-buttons-here"><p class="buttoni">Overview</p></button>

                        <button  style=" background-color: rgb(96, 26, 210)" id="barc-nav2" type="button" class="btn btn-default btn-lg overlay-buttons-here"><p class="buttoni">The potential of the idea</p></button>

                        <button  style=" background-color: rgb(96, 26, 210)" id="barc-nav3" type="button" class="btn btn-default btn-lg overlay-buttons-here"><p class="buttoni">Our light-induced systems</p></button>

                        <button style=" background-color: rgb(96, 26, 210)" id="barc-nav4" type="button" class="btn btn-default btn-lg overlay-buttons-here"><p class="buttoni">Experimental results</p></button>


                        <!-- Menu for tissue-->
                        <button style="background-color: rgb(49, 92, 247)" id="tissue-nav1" type="button" class="btn btn-default btn-lg overlay-buttons-here"><p class="buttoni">Overview</p></button>

                        <button style="background-color: rgb(49, 92, 247)" id="tissue-nav2" type="button" class="btn btn-default btn-lg overlay-buttons-here"><p class="buttoni">The potential of the idea</p></button>

                        <button style="background-color: rgb(49, 92, 247)" id="tissue-nav3" type="button" class="btn btn-default btn-lg overlay-buttons-here"><p class="buttoni">Our light-induced systems</p></button>

                        <button style="background-color: rgb(49, 92, 247)" id="tissue-nav4" type="button" class="btn btn-default btn-lg overlay-buttons-here"><p class="buttoni">Experimental results</p></button>


                        <!-- Menu for the bulb-->
                        <button style="background-color: rgb(246, 136, 80)"  id="bulb-nav1" type="button" class="btn btn-default btn-lg overlay-buttons-here"><p class="buttoni">Overview</p></button>

                        <button style="background-color: rgb(246, 136, 80)" id="bulb-nav2" type="button" class="btn btn-default btn-lg overlay-buttons-here"><p class="buttoni">Why a bacterial bulb</p></button>

                        <button style="background-color: rgb(246, 136, 80)" id="bulb-nav3" type="button" class="btn btn-default btn-lg overlay-buttons-here"><p class="buttoni">The biology</p></button>

                        <button style="background-color: rgb(246, 136, 80)" id="bulb-nav4" type="button" class="btn btn-default btn-lg overlay-buttons-here"><p class="buttoni">Taking it to the market</p></button>
                    </div>
                </div>

                <!-- This is where all the shit happens-->
                <div class="row">

                    <!-- Barchitecture bits-->
                    <div id="barc-1" class="text-section col-sm-11">

                        <p class="summary-here">Barchitecture is a 3D printing technology that uses light to guide cell adhesion between bacterial cells into 3D structures. Light also controls the production of biomaterials, enabling a final solid structure to be produced from cells. The end goal for our technology is to be a cheaper, more sustainable and eco-friendly alternative for construction/architecture and to demonstrate the usefulness of biological light switches in approaching challenges such as forming structures on Mars, quickly building temporary shelters, printing biocompatible medical implants and achieving ‘living buildings’.</p>

                        <img style="height: 60%; width:60%; margin:0 0 1% 20% !important" src="http://2017.igem.org/wiki/images/7/70/3D-printing.png" class="image-content-hidden">

                        <p class="text-content-new">Barchitecture works similarly to stereolithography, with cells acting as the prime material. Our design involves two suspended colonies of <i>E. coli</i> cells that, upon light exposure, will form a strong covalent bond between the two binding partners (SpyTag or SpyCatcher) displayed on the surface membrane. With light, we can control in a spatiotemporal manner cell adhesion, allowing for the formation of a 3D structure. For biopolymer production, light can also induce the accumulation of PHB granules inside the cells and their secretion. Once secreted, the granules bind the cell surface and are cross-linked into a solid shape by a photoinitiator.</p>

                        <p class="text-content-new">To achieve such as technology, we focused on the first steps: designing and testing a cell adhesion system that can be activated with light. We encourage future iGEM teams to look into optimizing the current designs and approaches to PHB production, secretion, binding and cross-linking ().</p>


                        <p class="text-content-new"> Top 5 reasons to use cells for structure generation:
                            (1) Cells span multiple length scales enabling the generation of microscopic to macroscopic structures. (2) Structures can be directed to morph, self-heal or be easily recycled by light-inducing cell death or polymer degradation. (3) No (or minimal) requirement for material mining, a costly and environmentally damaging activity. (4) It allows the generation of structures in environments that humans have no direct access to or are life-endangering (LINK TO IrrE AND ESA). (5) Provides more autonomy to designers/architects and opens up new innovative properties (be it cell-free or “living” structures) </p>


                    </div>

                    <div id="barc-2" class="col-sm-11">

                        <h1 style="text-align: center; padding:3% 7% 2% 0 !important;" class="title">The potential of Barchitecture</h1>

                        <p class="text-content-new">We researched the potential of our Barchitecture application to determine the end goal of applying biological light switches to bacterial cell adhesion and biopolymer production. Our <a style="text-decoration: none !important" href="http://2017.igem.org/Team:UCL/HP/Gold_Integrated">Integrated Practices</a> describe our conversations with architects, ARUP - the construction company, Blue Alpha - a PHB producing company and the European Space Agency, from which we designed our Architecture technology to address the following real-world issues.</p>

                        <div style="margin-left: 8% !important; background-color: rgba(32,88,137,0.12)" class="col-sm-3 application-boxes">

                            <p class="applications-text">Natural disasters</p>

                            <button id="barc-app1" type="button" class="btn btn-default btn-sm buttons-here">
                                <svg version="1.1" id="layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
                                     width="15%" height="15%" viewBox="0 0 162 115" enable-background="new 0 0 162 115" xml:space="preserve">
                                <g>
                                    <defs>
                                        <rect id="SVGID_1_" width="162" height="115"/>
                                    </defs>
                                    <clipPath id="SVGID_2_">
                                        <use xlink:href="#SVGID_1_"  overflow="visible"/>
                                    </clipPath>
                                    <g clip-path="url(#SVGID_2_)">
                                        <path d="M80.913,90.145l63.957-89.766h16.464l-80.42,112.914L0.492,0.379h16.464L80.913,90.145z"/>
                                    </g>
                                </g>
                                </svg>
                            </button>
                        </div>


                        <div style="background-color: rgba(137,2,19,0.12)" class="col-sm-3 application-boxes">

                            <p class="applications-text">Space Exploration</p>

                            <button id="barc-app2" type="button" class="btn btn-default btn-sm buttons-here">
                                <svg version="1.1" id="layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
         width="15%" height="15%" viewBox="0 0 162 115" enable-background="new 0 0 162 115" xml:space="preserve">
    <g>
        <defs>
            <rect id="SVGID_1_" width="162" height="115"/>
        </defs>
        <clipPath id="SVGID_2_">
            <use xlink:href="#SVGID_1_"  overflow="visible"/>
        </clipPath>
        <g clip-path="url(#SVGID_2_)">
            <path d="M80.913,90.145l63.957-89.766h16.464l-80.42,112.914L0.492,0.379h16.464L80.913,90.145z"/>
        </g>
    </g>
    </svg>
                            </button>
                        </div>


                        <div style="background-color: rgba(15,137,0,0.12);" class="col-sm-3 application-boxes">
                            <p class="applications-text">Medical implants</p>

                            <button id="barc-app3" type="button" class="btn btn-default btn-sm buttons-here">
                                <svg version="1.1" id="layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
         width="15%" height="15%" viewBox="0 0 162 115" enable-background="new 0 0 162 115" xml:space="preserve">
    <g>
        <defs>
            <rect id="SVGID_1_" width="162" height="115"/>
        </defs>
        <clipPath id="SVGID_2_">
            <use xlink:href="#SVGID_1_"  overflow="visible"/>
        </clipPath>
        <g clip-path="url(#SVGID_2_)">
            <path d="M80.913,90.145l63.957-89.766h16.464l-80.42,112.914L0.492,0.379h16.464L80.913,90.145z"/>
        </g>
    </g>
    </svg>
                            </button>

                        </div>

                        <div style="margin-left: 8% !important; border:solid 1px rgba(39, 32, 34, 0.16); height:397px; overflow: scroll !important;" class="col-sm-9">
                            <p id="applications-description">Considering the increasing frequency of natural disasters, having a cheap and immediate way to generate structures could be used to temporarily shelter people, seal leaks, or bridge over flooded areas. Microorganisms can be also engineered to simultaneously detect and/or feed off hazardous waste and help clean up septic systems. To this end, biological light switches would prove useful in having a way of precisely guiding the phenotypes desired from the bacteria.</p>
                        </div>


                    </div>

                    <div id="barc-3" class="col-sm-11">

                        <!-- Cell Adhesion-->
                        <div style="margin-left:2% !important" class="text-section col-sm-11">

                            <h1 class="title">Light control of cell adhesion for 3D structure formation</h1>

                            <p class="text-content-new">We developed (Fig1A) transcriptional and (Fig1B) post-translational light switches for the induction of cellular aggregation. The final engineered cells would be able to respond to the light stimuli and assemble into desired structures that can simultaneously produce and bind a desired polymer.</p>

                            <img style="height: 60%; width:60%; margin:0 0 0 17% !important" src="http://2017.igem.org/wiki/images/f/f5/AB-adhesion.png" class="image-content-hidden">

                            <p class="text-figure">Figure 1. Light induction of cellular aggregation. A) Depicts blue-light inducible expression system of proteins involved in cell-cell adhesion. B) Depicts UV-light activation of proteins involved in cell adhesion. In both cases, SpyTag and SpyCatcher are fused to the N-terminal of a truncated version of  intimin, an outer membrane surface protein for their cell surface display. Upon light induction, the aspartate residue in SpyTag and the lysine residue in SpyCatcher will spontaneously form strong and irreversible isopeptide covalent bonds, thus inducing cell aggregation.</p>

                        </div>

                        <div style="margin-left:2% !important" class="text-section col-sm-11">

                            <h1 class="title">A. Transcriptional light activation of cell adhesion</h1>

                            <p class="text-content-new">Using the blue light inducible promoter PBlind , we aim to use blue light (6hr, 475nm) to induce the transcription of a truncated version of Intimin (denoted intimin'), a cell surface protein, fused to either SpyTag (13 amino acids) or SpyCatcher (138 amino acids, 15kDa).</p>

                            <p class="text-content-new">SpyTag and SpyCatcher are two binding partners that come from CnaB2 (immunoglobulin-like collagen adhesin domain) of the FbaB protein, found in the invasive strains of S. pyogenes. SpyTag contains a reactive aspartate that forms a strong isopeptide bond with a reactive lysine residue in SpyCatcher when in close proximity.</p>

                            <img style="height: 60%; width:60%; margin:0 0 0 15% !important" src="http://2017.igem.org/wiki/images/3/3c/Intimin-fusions.png" class="image-content-hidden">
                            <p class="text-figure">Figure 1. Cell surface display of SpyTag and SpyCatcher. Intimin, a cell surface protein is truncated at its N terminal onto which SpyTag or SpyCatcher is fused.</p>
                            <p class="text-figure"><i>Design note:</i> after analysing multiple cell surface display proteins, we opted for Intimin (EaeA intimin) as it has shown to display passenger proteins of similar size to SpyCatcher and it had not been characterised by iGEM teams before. Other cell candidates considered include: ice nucleation protein, PgsA and OmpC ().</p>

                            <p class="text-content-new">For blue-light induction of cell adhesion, we designed 2 composite parts to be introduced into <i>E. coli</i> cells. Cell harbouring these constructs will display either SpyCatcher or SpyTag on the surface membrane with the help of the Intimin protein upon blue-light exposure</p>
                            <p class="text-content-new">Once a mixture of cells display SpyTag and SpyCatcher on their cell surface, the cells are expected to aggregate. Similar to stereolithography, structures would be generated layer by layer from a flat surface by exposing a particular radius with blue light.</p>

                            <!-- the 2 briobricks-->
                            <img src="http://2017.igem.org/wiki/images/a/ae/BB-Intimin-SpyTag.png" class="image-content-hidden">

                            <img src="http://2017.igem.org/wiki/images/d/dc/BB-Intimin-SpyCatcher.png" class="image-content-hidden">


                            <p class="text-content-new"> Pblind, our blue light inducible promoter is activated by EL222 (BBa_K2332004) upon blue-light exposure (465nm). Pblind will only allow RNAP to transcribe genes downstream upon blue-light (465) exposure, as this will induce a conformational change in EL222, a natural photosensitive DNA-binding protein, required for RNAP recruitment and transcriptional activation. To enable blue-light dependent transcriptional control, EL222 should be constitutively expressed by the cells. Since light can be controlled easily in space, time and degree, this new basic part will enable tight spatiotemporal control of gene expression, which in turn enables the tight regulation of cell adhesion required for intricate structure formation.</p>

                            <p class="text-content-new"> Placing Intimin'-SpyTag/SpyCatcher under the control of Pblind along with the constitutive expression of (EL222), will allow the transcriptional control of the cell surface display of SpyTag through blue-light (465nm). As shown in figure 2, Pblind consists of a fusion of the EL222 DNA binding region and the LuxI promoter (see figure 2). The lux box, a 20bp inverted repeat (LuxR and 3-oxo-C6-HSL complex binding region) from the luxI promoter, was replaced with the 18bp DNA binding region of EL222. In the dark, EL222 is inactive as its N-terminal LOV domain represses its DNA-binding C-terminal HTH domain. Upon blue light exposure (465nm), LOV-HTH interaction is released, allowing it to dimerize and bind its binding region, overlapping the -35 region of the luxI promoter. This ultimately results in the recruitment of RNAP and transcriptional activation. </p>

                            <img style="height: 60%; width:60%; margin:0 0 0 15% !important" src="http://2017.igem.org/wiki/images/archive/5/52/20171026152315%21Pblind-circuit.png" class="image-content-hidden">
                            <p class="text-figure">Figure 2. Genetic circuit for transcriptional light induction of cell aggregation. Represents  PBlind, blue-light inducible promoter upstream of intimin'-SpyTag and intimin'-SpyCatcher fusion proteins. </p>


                            <h1 class="title">Experimental design</h1>

                            <p class="text-content-new">Two cell lines expressing either intimin’-SpyTag or intimin’-SpyCatcher under the control of PBlind would have been grown overnight in darkness or under blue light (465nm) and either been mixed together, mixed with WT cells (control) (15min) or not mixed in the presence or absence of the EL222 construct. Cell mixtures would have then be passed through a particle sizer with a range of aperture sizes (0.4µm - 40µm) to compare aggregate sizes.</p>
                            <p class="text-content-new">As secondary experiments and for visual purposes we would have also incubated cells expressing Intimin-SpyTag with purified GFP-SpyCatcher, Intimin’-SpyCatcher with purified GFP-SpyTag and Intimin-GFP-SpyTag with Intimin-SpyCatcher cell lines for 15min pre and post light induction to visualize cell-protein binding and cell-cell interaction via fluorescence microscopy. We also included a His-Tag within all SpyCatcher construct variants to enable in-vitro analysis of complexes formed with SpyTag variants.</p>


                        </div>


                        <!-- Light Induced Control-->
                        <div style="margin-left:2% !important" class="text-section col-sm-11">

                            <h1 class="title">B. Post-translational light control of cell adhesion</h1>

                            <p class="text-content-new">As a quicker induction of cell aggregation, we aimed to develop a post-translational light-induced activation system in which cells are constitutively expressing either Intimin’ fusion proteins with SpyTag or an inactive version of SpyCatcher on their cell surface that can only bind cells expressing intimin'-SpyTag upon UV light (365nm) exposure (see Figure 1). SpyTag (13 amino acids) and SpyCatcher (138 amino acids, 15 kDa) form covalent isopeptide bonds and originate from CnaB2 (immunoglobulin-llike collagen adhesin domain) of the FbaB protein, found in the invasive strains of S. pyogenes.</p>

                            <p class="text-content-new">Only upon light exposure, the photocaged SpyCatcher will be able to bind SpyTag resulting in faster response than transcriptional induction of cell adhesion.</p>

                            <img src="http://2017.igem.org/wiki/images/7/7d/Intimin-catcher-caged.png" class="image-content-hidden">
                            <p class="text-figure">Figure 1: Post-translational light control system for cell aggregation. Cells are constitutively expressing an inactive photocaged version of SpyCatcher on their cell surface that can only bind cells expressing intimin'-SpyTag upon UV light (365nm) photolysis.</p>

                            <p class="text-content-new">This is achieved by incorporating a photocaged unnatural amino acid (UAA), Ne-methyl-L-lysine (see figure 2), in place of the reactive lysine in SpyCatcher required for the covalent bond formation with the SpyTag aspartate residue. Upon exposure to UV light (20min, 365nm), the “cage” group in the unnatural photocaged amino acid is cleaved off revealing the native amino acid and a biologically active protein. This approach also adds a layer of bio-containment as cells will only function when externally supplied with UAA.</p>

                            <img src="http://2017.igem.org/wiki/images/0/08/Photocaged-lysine.png" class="image-content-hidden">
                            <p class="text-figure">Figure 2: Photocaged unnatural amino acid (UAA), Ne-methyl-L-lysine.</p>


                            <p class="text-content-new"> As depicted in figure 3, we introduced an amber stop codon (TAG) in place of the reactive Lys 31 residue (Lys31X) in SpyCatcher fused to Intimin'. For this construct to work, amberless <i>E. coli</i> cells also have to express pyrrolysyl tRNA (pyIT/tRNAPylCUA) (BBa_K1223014) and pyrrolysyl-tRNA synthetase (BBa_K1223013) and the UAA, Ne-methyl-L-lysine, must be supplemented in the media (see Figure 2). The pyrrolysyl-tRNA synthetase catalyses the acylation of the suppressor tRNACUA with the UAA. During translation, the UAG amber codon in the mRNA is recognized by the acylated tRNACUA and the UAA will be added to the growing polypeptide chain. The orthogonality of this system has shown to work in both <i>E. coli</i> and mammalian cells [5,6].</p>


                            <img src="http://2017.igem.org/wiki/images/7/78/Photocaging-mechanism.png" class="image-content-hidden">

                            <p class="text-figure">Figure 3: Photocaging mechanism in <i>E. coli</i> cells. Amberless cells must be transformed with plasmids encoding tRNA-Pyl, Pyrrolysyl-tRNA Synthetase and the target protein with an amber codon at the position where the photocaged lysine residue (Ne-methyl-L-lysine) is to be introduced. Figure adapted from Mitra N. (2013). </p>


                            <h1 class="title">Experimental design</h1>


                            <p class="text-content-new">Amberless <i>E. coli</i> cells harbouring plasmids for the expression of pyIT (BBa_K2332003), pyrrolysyl-tRNA synthetase (BBa_K1223013) and the photocaged intimin’-SpyCatcher (BBa_K2332015) would have either been mixed with cells constitutively expressing Intimin’-SpyTag (BBa_K2332050) or mixed with WT cells (control) or not mixed and exposed to UV light (365nm) for 25min-1hr or not exposed and then passed through a particle sizer with a range of aperture sizes (0.4µm - 40µm) to compare aggregate sizes. </p>

                            <p class="text-content-new">As secondary experiments and for visual purposes we would have also incubated cells expressing Intimin’-SpyTag with purified photocaged GFP-SpyCatcher, photocaged Intimin’-SpyCatcher with purified GFP-SpyTag and Intimin-GFP-SpyTag with photocaged Intimin-SpyCatcher cell lines pre and post photo-lysis (365nm) for 25min-1hr to visualize cell-protein binding and cell-cell interaction via fluorescence microscopy. We also included a His-Tag within all photocaged SpyCatcher construct variants to enable in-vitro analysis of complexes formed with SpyTag variants.</p>

                        </div>




                        <!-- Polymer production/binding-->
                        <div style="margin-left:2% !important" class="text-section col-sm-11">
                            <h1 class="title">3. Polymer production/binding</h1>

                            <p class="text-content-new">Bacteria has been widely used for the production of biomaterials to generate sustainable and eco friendly bricks, bioplastic products, items of clothing as well as “living materials” by incorporating nanoparticles into biofilms. With Barchitecture we can light-induce cellular 3D structural arrangements while producing, binding or degrading biomaterials such as PHA or silicates.</p>

                            <p class="text-content-new">As shown in figure 1, PHA granules are produced through the expression of phaC1, phaA and phaB1 genes (BBa_K934001).</p>

                            <p class="text-content-new">The cells are also expressing a fusion protein of phasin (phaP1), a PHA binding protein, fused to HlyA C termini that will target the PHA granule for Type I secretion (encoded by HlyB and D) [2]. This results in PHA production, PHA binding by PhaP1-HlyA and its secretion outside the cell without lysing the cell. Therefore this results in constant PHA production and secretion in one step, which is more efficient and cost-effective than current microbial PHA production and extraction approaches.</p>
                            <p class="text-content-new">Additionally, our Barchitecture cells present phaP1 on their surface, by fusing it to outer membrane proteins (Ice nucleation protein) in order to bind the PHA secreted by the PHA producing/secreting cells.</p>

                            <img style="height: 60%; width:60%; margin:0 0 0 15% !important" src="http://2017.igem.org/wiki/images/5/5e/PHA-secretion-binding.png">
                            <p class="text-figure"> Figure 1. Proposed design for the production, secretion and binding of PHA granules. Cells express phaC1, phaA and phaB1 (BBa_K934001) for PHA granule production, PhaP1-HlyA, HlyB, HlyD for PHA granule binding and extracellular secretion, and INP-PhaP1 for granule binding on the cell surface.</p>

                            <p class="text-content-new">Through a similar mechanism, bacteria, could be light-guided to form precise and intricate structures that can then simultaneously produce and bind biosilica. This approach will enable “growing” electronics just with the guidance of a particular wavelength of light. Our engineered cells would form the desired 3D structure using blue light. These cells would also express photocaged (inactive) recombinant silicatein on their surface. Another wavelength of light would activate silicatein to begin the production of biosilica from water-soluble biosilica precursors added to the media. Our cells could also present ligands on their surface to attach the produced polymer to their surface.</p>

                        </div>

                    </div>

                    <div id="barc-4" class="col-sm-11">
                        <!-- Actual results-->
                        <div class="text-section col-sm-11">
                            <p class="title">Our wet lab data</p>

                            <p class="text-content-new">We tested whether the Pblind promoter has any significant leakage. Also, we wanted to show that GFP cannot be expressed in the absence of EL222. This is of particular interest as the aim of LIT is to demonstrate the versatility and high precision of light control. </p>

                            <p class="title">Experimental setup</p>

                            <p class="text-content-new">10-beta E. coli cells were transformed with GFP constructs under the control of different promoters: J23151-GFP (positive control), R0040-GFP (negative control), Pblind-GFP or not transformed (WT). J23151 is a constitutive promoter, R0040 is a TetR repressible promoter (repression inhibited only by the addition of tetracycline), Pblind promoter is a fusion of EL222 (photosensitive transcription factor) binding region and the luxI promoter, where EL222 is only able to dimerize and bind the Pblind promoter upon blue light exposure, where it can then recruit RNAP and drive the transcription of genes downstream. Colony PCR was used to identify successful transformants. For each construct, 3 of the identified colonies were aliquoted into 5ml of growth media (LB media with 25ug/ml Chloromphenicol), and grown overnight at 37°C in darkness (covered in aluminium foil) or exposed to blue light (465nm). Each biological replicate was then diluted in LB to OD600=0.6. 200μLx4 of each biological replicate were aliquoted into a black flat bottom 96 well plate. LB media with 25ug/ml Chloromphenicol was also aliquoted for fluorescence baseline determination. The fluorescence of all repeats along with the LB was measured using a Tecan Safire 2 Multi-Mode Plate Reader.</p>

                            <p class="title">Results and Analysis</p>

                            <img src="http://2017.igem.org/wiki/images/1/1b/Results-graph-barc.png" class="image-content-hidden">

                            <p class="text-figure">Figure 1. Blue light inducible promoter (Pblind) characterisation. J23151 is a constitutive promoter, R0040 is a TetR repressible promoter (repression inhibited only by the addition of tetracycline), Pblind promoter is a fusion of EL222 (photosensitive transcription factor) binding region and the luxI promoter, where EL222 is only able to dimerize and bind the Pblind promoter upon blue light exposure, where it can then recruit RNAP and drive the transcription of genes downstream. </p>

                            <p class="text-content-new">This construct allowed us to test whether the promoter Pblind has any significant leakage. We wanted to show that GFP cannot be expressed in the absence of EL222. This is of particular interest as the aim of LIT is to demonstrate the versatility and high precision of light control. As shown in Figure 1, only J23151-GFP (positive control) had a significant difference in fluorescence compared to R0040-GFP (negative control), Pblind-GFP, WT cells and the Luria Broth (LB) in both dark and Blue-light conditions. Pblind-GFP had no significantly different fluorescence level compared to the LB baseline, negative control or WT cells in either condition. This is expected, as the EL222 protein is required for blue-light inducible transcriptional activation.</p>

                        </div>



                    </div>

                    <!-- Tissue engineering bits-->

                    <!-- Overview-->
                    <div id="tissue-1" class="text-section col-sm-11">

                        <p class="text-content-new">Our guidance system for mammalian tissue engineering uses light activated proteins and genes that control cellular behaviour - an approach that can replace common tissue engineering approaches by overcoming current challenges.</p>

                        <p class="text-content-new">To use optogenetics in controlling cellular behaviour, several challenges need to be addressed. We analysed natural mechanisms of tissue development and found different entry points at certain stages in the development. Then, we designed our optogenetic guidance system to hijack the natural systems with minimal changes to the underlying biochemical pathways. In the cells of growing tissue, these are the mechanisms to be controlled through light:</p>

                        <div style="margin-left: 10% !important; background-color: rgba(32,88,137,0.12)" class="col-sm-4 application-boxes">

                            <p class="applications-text">Instantaneous cell adhesion</p>

                            <button id="tissue-app4" type="button" class="btn btn-default btn-sm buttons-here">
                                <svg version="1.1" id="layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
         width="15%" height="15%" viewBox="0 0 162 115" enable-background="new 0 0 162 115" xml:space="preserve">
    <g>
        <defs>
            <rect id="SVGID_1_" width="162" height="115"/>
        </defs>
        <clipPath id="SVGID_2_">
            <use xlink:href="#SVGID_1_"  overflow="visible"/>
        </clipPath>
        <g clip-path="url(#SVGID_2_)">
            <path d="M80.913,90.145l63.957-89.766h16.464l-80.42,112.914L0.492,0.379h16.464L80.913,90.145z"/>
        </g>
    </g>
    </svg>
                            </button>
                        </div>

                        <div style="margin-right:10% !important; background-color: rgba(137,2,19,0.12)"  id="current" class="col-sm-4 application-boxes">

                            <p class="applications-text">Gene activation</p>

                            <button id="tissue-app5" type="button" class="btn btn-default btn-sm buttons-here">
                                <svg version="1.1" id="layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
         width="15%" height="15%" viewBox="0 0 162 115" enable-background="new 0 0 162 115" xml:space="preserve">
    <g>
        <defs>
            <rect id="SVGID_1_" width="162" height="115"/>
        </defs>
        <clipPath id="SVGID_2_">
            <use xlink:href="#SVGID_1_"  overflow="visible"/>
        </clipPath>
        <g clip-path="url(#SVGID_2_)">
            <path d="M80.913,90.145l63.957-89.766h16.464l-80.42,112.914L0.492,0.379h16.464L80.913,90.145z"/>
        </g>
    </g>
    </svg>
                            </button>
                        </div>

                        <div style="margin-right: 10% !important; margin-bottom: 5% !important; margin-left: 10% !important; border: solid 1px rgba(39, 32, 34, 0.16); height:397px; overflow: scroll !important;" class="col-sm-8">
                            <p id="applications-tissue-description2">In order to form structures cells need to form dynamic non-covalent formations in adherens junctions that allow constant remodelling of the structure while still maintaining a high degree of rigidity. The cell adhesion proteins in these junctions are stabilized via connections with the cytoskeleton, most notably actin filaments. Cells forming synthetic tissues therefore need to have the ability to form these connections rapidly in a cell culture that is dividing. In mammalian cells transcription takes around 30 min, translation (incl. mRNA export) another 30 min and further protein sorting and processing is dependent on the the type of protein. For the purpose of forming structural patterns, a photosensitive cell adhesion system therefore needs to be post-translational and instantaneous.</p>

                            <ul id="hidden-list"><p class="text-content-new">The ones that are most important for tissue engineering are:</p>

                                <li class="list-item-content">Dedifferentiation</li>
                                <li class="list-item-content">Differentiation - Tissue consists of a range of specialised cells and their products and there is never a type of tissue made up of only one cell. To form a type of tissue cells, therefore, have to differentiate into a range of different cell types. This process requires already existing pluripotent stem cells. But in order to make this process feasible in healthcare current methods of inducing cells into a pluripotent state a too inefficient. We therefore propose that a gene activation system is needed that can induce dedifferentiation and differentiation triggered by different light sources.</li>
                                <li class="list-item-content">Phototaxis - Next to forming connections between cells and rigidifying these by consecutive establishment of an ECM, cells need to migrate inside and on top of the tissue. One of the major disadvantages of current tissue engineering approaches is that a cell’s movement once seeded onto a scaffold is hard to control. However, if this movement is guided by a light-path more control can be gained for the structuring process.</li>
                                <li class="list-item-content">Proliferation</li>
                                <li class="list-item-content">Apopotosis - A growing tissue is defined by a dynamic interplay between cellular proliferation and death. For instance, the human embryo originally has webbed feet and hands but the controlled apoptosis of the interdigital regions is responsible for the development of our fine tuned fingers. Similar, in a developing tissue we want to control cell death in certain regions while leaving others to continue proliferating.</li>

                            </ul>
                        </div>

                        <p style="padding-top: 10% !important" class="text-content-new">We first developed a photosensitive form of the common cell adhesion protein E-cadherin for structure formation, inspired by human body cells that first form structures and then develop into specialised cells. To control the second stage, where cells differentiate (i.e. become specialised via up- and down-regulation of certain genes), we designed a dCas9-TF system, attached to the plasma membrane via the photocleavable linker PhoCl for specific gene activation. When PhoCl is cleaved by light, the dCas9-TF is free to move into the nucleus and activate the gene of interest. In comparison to other approaches, our system would require only a brief pulse of light to cleave PhoCl and be activated.</p>


                        <p class="text-content-new">A modified multi-photon microscope, using two light-sources to stimulate photosensitive proteins at the intersection of the laser lights, would give the system not only temporal but also spatial boundaries. Our choice for such a microscope would not require long exposure times and ensure specificity of photoactivation, compared to current light-induced methods.</p>


                    </div>

                    <!-- Challenges to resolve-->
                    <div style="margin-left: 3% !important" id="tissue-2" class="text-section col-sm-11">

                        <h1 style="text-align: center !important; padding: 3% 0 1% 0 !important;" class="title">Why tissue engineering & why optogenetics?</h1>

                        <p class="text-content-new">Tissue engineering allows the production of tissue structures that are specific to somebody's own cell and has the potential to overcome many of the problems seen in human-to-human transplantation.</p>

                        <ul style="margin-bottom: 3% !important; margin-left: 10% !important; margin-right: 5% !important" class="list-section">
                            <p style="padding-left: 40% !important" class="title">Advantages</p>
                            <li style="text-align: justify; padding: 3% 10% 0 5% !important" class="list-item-content">Using a patient’s own cells to create such tissues ensures there will be <span style="color: #ff6657">no rejection</span> by the host</li>
                            <li style="text-align: justify; padding: 3% 10% 0 5% !important" class="list-item-content">Independent of long transplant waiting lists <span style="color: #6499ff">patients would not suffer</span> the impaired living conditions and death due to a failing organ</li>
                            <li style="text-align: justify; padding: 3% 10% 0 5% !important" class="list-item-content">Streamlining the production of tissues in vitro would <span style="color: #ff6657">reduce the costs</span> and <span style="color: #5f9eff">enable access</span> to transplantations to people living in underprivileged conditions</li>

                        </ul>

                        <p class="text-content-new">To achieve this vision, our project addresses common problems found in tissue engineering, by constructing a molecular tool that allows the control of cells while they are forming a tissue through light. Typically, sophisticated 3-dimensional scaffolds that mimic the tissue are developed to try to reproduce qualities such as elasticity, volume and organization. After the scaffold has been produced, cells are seeded into and onto the scaffold and they are allowed to grow into the scaffold.</p>

                        <p style="position:relative; text-align: center; padding-top:2% !important; padding-bottom: 3% !important" class="text-content-new">However, the scaffolding and cell-seeding method, often called ‘bioprinting’ or ‘cellular 3D-printing’ has many drawbacks:</p>


                        <div style="margin-left: 10% !important; background-color: rgba(32,88,137,0.12)" class="col-sm-3 application-boxes">

                            <p class="applications-text">Choosing the right scaffold</p>

                            <button id="tissue-app1" type="button" class="btn btn-default btn-sm buttons-here">
                                <svg version="1.1" id="layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
         width="15%" height="15%" viewBox="0 0 162 115" enable-background="new 0 0 162 115" xml:space="preserve">
    <g>
        <defs>
            <rect id="SVGID_1_" width="162" height="115"/>
        </defs>
        <clipPath id="SVGID_2_">
            <use xlink:href="#SVGID_1_"  overflow="visible"/>
        </clipPath>
        <g clip-path="url(#SVGID_2_)">
            <path d="M80.913,90.145l63.957-89.766h16.464l-80.42,112.914L0.492,0.379h16.464L80.913,90.145z"/>
        </g>
    </g>
    </svg>
                            </button>
                        </div>

                        <div style="background-color: rgba(137,2,19,0.12);" class="col-sm-3 application-boxes">

                            <p class="applications-text">Cellular Organisation</p>

                            <button style="margin-bottom:5% !important" id="tissue-app2" type="button" class="btn btn-default btn-sm buttons-here">
                                <svg version="1.1" id="layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
         width="15%" height="15%" viewBox="0 0 162 115" enable-background="new 0 0 162 115" xml:space="preserve">
    <g>
        <defs>
            <rect id="SVGID_1_" width="162" height="115"/>
        </defs>
        <clipPath id="SVGID_2_">
            <use xlink:href="#SVGID_1_"  overflow="visible"/>
        </clipPath>
        <g clip-path="url(#SVGID_2_)">
            <path d="M80.913,90.145l63.957-89.766h16.464l-80.42,112.914L0.492,0.379h16.464L80.913,90.145z"/>
        </g>
    </g>
    </svg>
                            </button>
                        </div>

                        <div style="margin-right: 10% !important; background-color: rgba(15,137,0,0.12)" class="col-sm-3 application-boxes">
                            <p class="applications-text">Controlling differentiation</p>

                            <button id="tissue-app3" type="button" class="btn btn-default btn-sm buttons-here">
                                <svg version="1.1" id="layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
         width="15%" height="15%" viewBox="0 0 162 115" enable-background="new 0 0 162 115" xml:space="preserve">
    <g>
        <defs>
            <rect id="SVGID_1_" width="162" height="115"/>
        </defs>
        <clipPath id="SVGID_2_">
            <use xlink:href="#SVGID_1_"  overflow="visible"/>
        </clipPath>
        <g clip-path="url(#SVGID_2_)">
            <path d="M80.913,90.145l63.957-89.766h16.464l-80.42,112.914L0.492,0.379h16.464L80.913,90.145z"/>
        </g>
    </g>
    </svg>
                            </button>

                        </div>

                        <div style="margin-right: 10% !important; margin-bottom: 3% !important; margin-left: 10% !important; border:solid 1px rgba(39, 32, 34, 0.16); height:300px; overflow: scroll !important;" class="col-sm-9">
                            <p id="applications-tissue-description">Scaffold design and construction methods need to be tailored towards the tissue tried to be produced. Scaffolds for heart reconstruction need to fulfil completely different prerequisites to bone scaffolds. With the incredibly diverse microenvironment of cells a myriad of different designs have been tested. Furthermore, the rapid invention of new biomaterials that are usable as scaffold material increase the number of possible scaffolds even further</p>
                        </div>


                        <!-- Conclusion of why light and what should be done in the future-->


                        <h1  class="title">Why optogenetics can address these challenges</h1>

                        <p class="text-content-new">A combination of optogenetic genes and light stimulating these genes has the potential to allow precise spatiotemporal control of tissue generation at the cellular and even molecular level compared to other approaches.</p>

                        <p class="text-content-new">Previously, MIT iGEM 2010 used mechanosensors as triggers for cell differentiation based on the fact that mechanical stimuli play an important role during in vivo tissue formation. But as shown by MIT iGEM 2010 mechanical inputs rely on the physical form of the input and require physical penetration of the cells which is not possible for cells on the interior of a growing tissue.</p>

                        <p class="text-content-new">Typically chemical stimuli are used. Growth factors, morphogens and mitogens are used in iPS cell research to induce differentiation as mentioned above. Chemokines and cytokines are used for the recruitment of cells to certain areas in the body. However, chemical stimuli cannot differentiate between the cells that we want them to act on and others we don’t want them to work on. Off-target activation and differentiation of cells is quite common and therefore unsuitable for tissue generation.</p>

                        <p class="text-content-new">Optical stimuli in comparison to mechanical ones do not require physical stimulation. As shown through multi-photon microscopy, it is possible to activate single molecules on a molecular level through the simultaneous exposure to different light sources. This means optical stimuli have the potential to activate photosensitive proteins in space and time at a subcellular level on the in- and outside of a growing tissue.</p>


                        <p class="text-content-new"> In comparison to chemical stimuli, optical stimuli do not diffuse in solution and the risk of off-target cell activation is reduced.</p>

                        <p class="text-content-new">  Furthermore, for future standardization, optical stimuli do not require specific scaffolds to build on. However, they do require a one-time acquisition of light sources capable of dynamically controlling the optogenetic circuits inside the cells.</p>

                    </div>

                    <!-- Our designs-->
                    <div id="tissue-3" class="col-sm-11">

                        <div class="col-sm-11">
                            <p class="text-content-new">Clearly, the optogenetic guidance system of tissue engineering has well-founded potential. To achieve our vision, we focused on better characterising existing optogenetic tools in two scenarios: instantaneous cell adhesion and light-induced gene activation and regulation - key to achieving controlled tissue growth, as previously mentioned.</p>

                            <p class="text-content-new">Currently, in the synthetic biology community we need to resolve the need for more optogenetic tools and to characterise the existing ones better.</p>

                            <div style="margin-left:5% !important" id="instacell" class="text-section col-sm-11">
                                <h1 class="title">Instantaneous cell adhesion</h1>

                                <h1 style="color:#dc5162" class="subtitle">The problem</h1>

                                <p class="text-content-new">A synthetic tissue cannot rely on a stream of previous information in the same way that embryo cell adhesion does - in natural development, molecules and cells interact with each other in a complex system before and after cell-cell connections are formed. Our optogenetic approach would rather hijack these processes elegantly.</p>

                                <p class="text-content-new">The solution, then, is post-translation control. This approach enables us to combine light control with protein-protein inhibition, which in comparison to transcriptional control results in less noise and fewer off-target activation. Also, the process is instantaneous, allowing for light to break any prevention of cell adhesion immediately.</p>

                                <h1 style="color:#dc5162" class="subtitle">Design</h1>


                                <p class="text-content-new">First, we designed a BioBrick based on the preproprotein of E-cadherin (BBa_K2332312) and tested its capability of forming cell-cell adhesions via aggregation assays. Prof. Stephen Price, UCL, kindly provided us the E-cadherin preproprotein - a key element in how tissue cells adhere to each other naturally. Specifically, E-cadherin is a calcium-dependent cell adhesion molecule that functions in the establishment and maintenance of epithelial cell morphology during embryongenesis and adulthood. During the secretory pathway the encoded preproprotein undergoes proteolytic processing to generate a mature protein.</p>

                                <p class="text-content-new">Previously, iGEM UCSF 2011 used the extracellular domain of E-cadherin (BBa_K644000) trying to form cell connections. However, E-cadherin’s function depends not only on the presence of calcium but also on the bonding of linker-proteins like alpha and beta catenin to the cytosolic domain of E-cadherin and the actin filaments in the cortex of the mammalian cell. This ‘anchoring’ results in stable mass action and the formation of adherens junctions between cells.</p>

                                <p class="text-content-new">We chose the interaction between the cytosolic domain of E-cadherin and beta-catenin as an entry point into E-Cadherin’s physiology to render the protein photosensitive. By fusing the novel photocleavable protein PhoCl to the cytosolic domain of E-cadherin the interaction with beta-catenin is sterically inhibited. PhoCl is an engineered green-to-red photoconvertible fluorescent protein (Zhang et al. 2017) which we received as a gift from the Robert Campbell lab, University of Alberta, Canada. It consists of 232 amino acids that are cleaved into an ~66-residue N-terminal fragment and an ~166-residue C-terminal fragment. We designed a new BioBrick based on PhoCl (BBa_K2332311) and a BioBrick of the E-cadherin-PhoCl fusion protein (BBa_).</p>

                                <p class="text-content-new">The fusion protein cannot form stable cell-cell connections due to the lack of interactions with the actin cortex. However, after 400 nm, violet light exposure the sterical interference of the N-terminal 66 amino acid PhoCl remnant is too weak to inhibit the formation of connections between E-cadherin and the actin cortex. Stable connections can now be formed between cells and photo-activation is achieved.</p>

                                <p class="text-content-new">A two-step experimental approach was chosen: first test the E-cadherin preproprotein via an aggregation assay for the ability to form patterns and then test the E-cadherin-PhoCl fusion for photo-sensitivity.</p>
                            </div>


                            <div  style="margin-left:5% !important" id="geneactiv" class="text-section col-sm-11">
                                <h1 class="title">Light-induced Gene Activation & Regulation</h1>

                                <h1 style="color:#dc5162" class="subtitle">The problem</h1>

                                <p class="text-content-new">We considered different approaches that have been tried to activate genes via light to obtain precise and reliable gene activation. For example, Motta-Mena et al. 2014 used a for mammalian cells engineered version of the EL222 protein, a bacterial light-oxygen-voltage protein (further described in the Barchitecture module) to activate genes in a range of mammalian cell lines. However, this design requires the upstream positioning of the associated promoter in front of the gene of interest. This obviously limits its use in our proposed guidance system which tries to hijack the organism’s own genome.</p>

                                <p class="text-content-new">Therefore, many researchers developed several optogenetic transcriptional control systems for dynamically regulating gene expression with light.</p>

                                <p class="text-content-new">Most of these systems use light-inducible heterodimerizing proteins from plants. For instance, fusion of one heterodimerizing protein to a ZFP18 or TALE4 and fusion of its binding partner to a transcriptional activation domain, such as VP64, enables light-dependent recruitment of the activation domain to the DNA sequence targeted by the ZFP or TALE. These systems enable control over expression of any gene in a reversible, tunable and spatially defined manner. However, reengineering the ZFP or TALE DNA-binding domain to target new sequences can be laborious and require specialized expertise. This is particularly a concern for gene activation with systems that must target several sequences in a gene promoter to synergistically achieve robust activation.</p>


                                <p class="text-content-new">To address the limitations in other approaches based on light, researchers adapted the CRISPR-Cas9 activator system for optogenetic control. Polstein et al. 2015 showed that fusing the light-inducible heterodimerizing proteins CRY2 and CIB1 from Arabidopsis thaliana to the VP64 transactivation domain and either the N- or C- terminus of dCas9, the catalytically inactive form of Cas9 (D10A, H840A) allows induction of transcription of endogenous genes in the presence of blue light.</p>

                                <p class="text-content-new">This light-activated CRISPR-Cas9 effector (LACE) system can easily be modified by changing the gRNA that is expressed inside the cell and therefore allows dynamic on- and off-switching of genes. Nihongaki et al. 2015 modified the LACE system by using the transactivation domain of NFκB, called p65, instead of VP64 and activated endogenous and exogenous genes in a spatiotemporally confined manner.</p>

                                <p class="text-content-new">However, these LACE systems are limited by the exposure time to blue light and require widefield illumination of the entire cell in order to activate both components, the dCas-CIB1 and the CRY2-TF. Gene activation in these systems requires constant blue light exposure for at least 2-3 hours for measurable results.</p>

                                <p class="text-content-new">In the context of controlling cells to form tissues, this dependence on widefield illumination presents a major problem. All cells in the tissue would be illuminated at the same time and no specificity could be achieved. It is therefore necessary to reduce the required exposure time for the activation of the LACE system and render the activation mechanism independent of widefield illumination.</p>


                                <h1 style="color:#dc5162" class="subtitle">Design</h1>


                                <p class="text-content-new">Our solution is a LACE system that does not depend on heterodimerizing proteins from plants but instead on post-translational control exerted through a photocleavable linker.</p>

                                <p class="text-content-new">The system consists of four essential units: a transmembrane domain, a photocleavable linker, dCas9 and a transactivation domain (p65, VP64, VP16).</p>

                                <p class="text-content-new">Based on these prerequisite we designed BioBrick BBa_K2332317, a single-illumination light-activated CRISPR-Cas9 effector (siLACE) system localised to the plasma membrane of the cell via the photocleavable linker PhoCl. By localising the system to the plasma membrane the dCas9 fused to the transactivation domain is preprogrammed to not enter the nucleus and can therefore not activate any target gene inside the nucleus.</p>

                                <p class="text-content-new">However, a single laser beam of 400 nm, violet light has been shown to be able to cleave PhoCl. After cleavage, the dCas9-TF is able to enter the nucleus, directed by attached nuclear localisation sequences. Inside the nucleus the system uses gRNAs to find the endogenous target gene and induce transcription.</p>

                                <p class="text-content-new">The illumination radius of such the laser is many magnitudes below the dimensions of a standard mammalian cell (diameter: 10-15 µm; volume: ~4,000 µm3). This means that our siLACE system is not only able to activate genes on a single cell level but even modulate the expression via subcellular activation of only certain plasma membrane areas.</p>

                                <p class="text-content-new">The main advantages of this siLACE system over previous designs is its independence of widefield illumination, its ‘memory’ and its ability for modulation. The system allows activation of genes solely via a single laser beam and once activated stays in an active state until the siLACE protein is naturally degraded. Additionally by changing the laser radius and the exposure time it is possible to activate different amounts of dCas9-TF’s at the same time, leading to short or long expression of the target gene.</p>

                                <h1 style="color:#dc5162" class="subtitle">Design considerations</h1>


                                <p class="text-content-new">For the construction and testing of our siLACE system we divided it into two testable components: the plasma membrane delivery system (PMDS, BBa_K2332315) and the dCas9-p65 (BBa_K2332316).</p>

                                <h1 style="color:#dc5162" class="title">Plasma membrane delivery system (PMDS)</h1>

                                <p class="text-content-new">The aim of the PMDS design was to create a generally practical transmembrane protein embedded in the plasma membrane that localises a target protein to its cytosolic side and allows subsequent delocalisation via light-induction.</p>

                                <p class="text-content-new">There are many commercially available expression vectors available that target recombinant proteins to the surface of mammalian cells. (e.g.  ThermoFisher’s pDisplay). However, our PMDS is not only supposed to target a protein to the outside of the PM but also to the inside. LIT therefore had to investigate membrane trafficking in mammalian cells in depth to ensure the most efficient design.</p>

                                <p class="text-content-new">In mammalian cells, all proteins start translation in the cytoplasm (with the exception of some mitochondrial proteins) and either undergo protein sorting during or after translation. PM transmembrane proteins first encode a signal peptide that is immediately bound by the signal recognition particle (SRP) which localises the translation complex to the endoplasmic reticulum (ER) where the translation continues through the Sec61-channel. During the co-translational import into the ER the presence or absence of special sequences of hydrophobic amino acids defines if a protein will contain one, two or multiple transmembrane domains or if the protein will be soluble (absence of hydrophobic sequence). If only one hydrophobic sequence is encoded the protein will be a single-span transmembrane protein. By default all proteins encoded enter the secretory pathway if they do not contain specialised signal sequences or signal patches, surface markers that define a proteins destination inside the cell.</p>

                                <p class="text-content-new">We therefore had to incorporate a well characterised ER signal peptide into our PMDS, ensure the translation of a stable transmembrane domain and be cautious to not include a special signal sequence into the design.</p>

                                <p class="text-content-new">iGEM LMU-TU Munich 2016 tested three different ER signal peptides of which ‘BM-40’ (BBa_K2170214) yielded the best result in a luciferase secretion assay. Unfortunately, this BioBrick is not in stock in the registry of standard biological parts and so we contacted iGEM  LMU-TU Munich 2017 who provided us with the plasmid.</p>

                                <p class="text-content-new">The next step was to find the most promising transmembrane domain for our PMDS. We conducted a bioinformatic analysis of different transmembrane domains from the registry as well as from Nagaraj et al. 2011 and Quadrat and Truong 2016. Both papers are from the Applied Protein Engineering lab, University of Toronto, and investigate strategies for the assembly of synthetic transmembrane proteins. The result of our analysis was that the TMD of EGFR (BBa_K2170210) would be most suitable for ensuring transmembrane anchoring and a defined orientation of the protein. </p>

                                <p class="text-content-new">The final design of the coding region of our PMDS is shown in Figure .. . It consists of the signal peptide ‘BM-40’, an ‘ectodomain insert’, the ‘TMDEGFR’, the photocleavable linker ‘PhoCl’, ‘cytosolic domain insert’ and stop codon (5’ to 3’).</p>

                                <p class="text-content-new">For our test construct we chose BBa_K648013, a GFP fused N-terminally to a FLAG epitope tag, followed by an enterokinase cleavage site, as our ‘ectodomain insert’. In doing so we further characterised BBa_K648013 because the entry from iGEM Penn State 2011 did not mention that the submitted BioBrick contains a FLAG epitope or enterokinase cleavage site. This BioBrick was chosen because the GFP allows for easy tracking of the protein during its synthesis and the secretory pathway. Additionally, the enterokinase cleavage site allows for easy orientation testing of our PMDS by adding enterokinase into the cell solution and recording delocalisation of the GFP from the PM into the solution.</p>

                                <p class="text-content-new">For the ‘cytosolic domain insert’ we chose the mCherry BioBrick BBa_J06504 followed by a NLS. mCherry was chosen because because its absorption maximum of 585 nm wavelength light lies outside the emission spectrum of GFP (max. At 510 nm) which makes makes separate imaging easier by reducing bleed-through.</p>


                                <h1 style="color:#dc5162; font-size: 1.6vw !important" class="subtitle">Experimental design</h1>

                                <p class="text-content-new">Previous iGEM teams have used a range of different methods for gene delivery into mammalian cells. Some chose viral delivery, others tried to tried to make the iGEM submission vector pSB1C3 a mammalian expression vector by adding a device for mammalian antibiotic expression and creating a device for the gene of interest with mammalian promoters and terminators. Both these processes take a long time and have a high risk of failure. This is why MIT 2010 proposed a new standard to concatenate mammalian parts based on Gateway cloning instead of restriction cloning, the MammoBlock standard. However, this approach requires buying additional material and requires modification of standard BioBricks for Gateway cloning. Since time is the most pressing issue during a one summer research project LIT decided to use standard mammalian expression plasmids instead. We chose to use pcDNA3.1, a standard mammalian expression plasmid, because it is already optimized for protein expression in mammalian cell lines and contains a 5’-UTR and 3’-UTR that is well characterized and ensures efficient translation of the gene of interest.</p>

                                <p class="text-content-new">After restriction cloning of our PMDS coding region into pcDNA3.1 we aim to transfect HEK293T cells by using Qiagen’s “SuperFect® Transfection Reagent” kit, a transfection method based on activated-dendrimers. This method has been successfully used for the aggregation assay (see above).</p>

                                <p class="text-content-new">Using a confocal microscope we are able to see localisation of GFP and mCherry to the plasma membrane. Using a 400 nm, violet laser we are able to induce photocleavage of specific cells in the culture and record the movement of mCherry from the PM to the nucleus. Quantitative data is produced through analysis of percentage colocalization of GFP and mCherry.</p>

                                <p class="text-content-new">This system can be tested even further in the future by using an mCherry part that contains a degradation tag (e.g. BBa_K1926013). This would allow a more dynamic system characterised by fast localisation of mCherry to the nucleus and consecutive degradation, returning the cell to its original state.</p>

                                <p class="text-content-new">Finally, we incorporated a KpnI cutting site between PhoCl and mCherry and a BamHI cutting site between mCherry and the stop codon. These restriction sites allow easy exchange of the mCherry part with any other gene flanked by these sites in an iterative plug-and-play fashion (see Litcofsky et al. 2012). These two enzymes were chosen because they are not found in the iGEM submission plasmid pSB1C3 and therefore allow the plug-and-play method with a wide range of BioBricks.</p>


                                <h1 style="color:#dc5162" class="subtitle">dCas9-TF</h1>

                                <p class="text-content-new">The aim of the dCas9-TF design was to replicate the results of Nihongaki et al. 2015 with a dCas9-p65 fusion protein instead of a dCas9-CIB1 and CRY2-p65 heterodimerizing system. For this purpose we contacted Dr. Nihongaki who gave us the sequences for his constructs as a gift.</p>

                                <p class="text-content-new">The dCas9-p65 is targeted towards an mCherry reporter via a gRNA that has 13 different binding sites upstream of the mCherry reporter.</p>

                                <p class="text-content-new">Using pcDNA3.1 as the vector and Qiagen’s “SuperFect® Transfection Reagent” kit as the transfection method we aim to express exogenous mCherry as reference for the testing of our final siLACE construct.</p>

                                <p class="text-content-new">The dCas9-p65 coding region has been flanked by a KpnI cutting site (5’-end, downstream of the start codon) and a BamHI cutting site (3’-end, upstream of the stop codon) for further usage.</p>


                                <h1 style="color:#dc5162" class="subtitle">siLACE system</h1>

                                <p class="text-content-new">After we produced and tested our two test constructs the final single-illumination light-activated CRISPR-Cas9 effector system (BBa_K2332317) is created by choosing the dCas9-p65 gene as the ‘cytosolic domain insert’ of the PMDS. Since the dCas9-p65 incorporates a 5’ KpnI and a 3’ BamHI site it is suitable for plug-and-play exchange with the mCherry from our test PMDS. PMDS is thereby modified to deliver the dCas9-p65 to the plasma membrane instead of mCherry.</p>

                                <p class="text-content-new">Using pcDNA3.1 as the vector and Qiagen’s “SuperFect® Transfection Reagent” kit as the transfection method we expect that no exogenous mCherry will be expressed after the transfection because the PMDS inhibits dCas9-p65 to enter the nucleus. Only after photoactivation with 400 nm, violet light will PhoCl be cleaved and irreversibly release the dCas9-p65 which now enters the nucleus. Inside the nucleus it associated with the gRNA and binds to the 13 different binding sites on the mCherry reporter. The transactivation domain p65 subsequently induces activation of transcription of the mCherry gene.</p>

                                <p class="text-content-new">The presented experiments are our designs which we could not conduct during the summer anymore because of time issues. However, we encourage future iGEM teams to use the designs we worked on over the summer and continue where we stopped (which counts as further characterisation of an existing BioBrick and is a Gold medal criteria).</p>


                                <h1 style="font-size: 1.6vw !important" class="subtitle">Extra notes on Design considerations</h1>


                                <p class="text-content-new">LACE systems can be modified in a wide variety of ways. As described above many approaches are based on the exchange of one transactivation domain with another (e.g. VP64 with p65). Transactivation domains activate gene transcription in a local fashion and generally don’t activate more than one downstream gene.</p>


                                <p class="text-content-new">A fundamentally different approach is the exchange of the transactivation domain with epigenome modifying proteins. Hilton et al. 2015 designed a dCas9 fused to the histone acetyltransferase (HAT) p300 and tested its ability to activate gene transcription for a range of genes in comparison to a dCas9-VP65 LACE system. The dCas9-p300 fusion protein catalyzes acetylation of histone H3 lysine 27 at its target sites, leading to robust transcriptional activation of target genes from promoters and both proximal and distal enhancers. This means that dCas9-p300 is able to induce transcription in proximal and distal genes up to 46 kb away from the dCas9 binding site. These results indicate the possibility of using epigenome modifying enzymes in LACE systems and its usefulness in our proposed guidance system for tissue engineering but they also point out the complex interplay of mammalian gene compaction and gene activation and how much we still have to learn to make our vision a reality.</p>

                                <h1 style="color:#dc5162" class="subtitle">Bioinformatics analysis</h1>

                                <p class="text-content-new"><b>Aim:</b> construct a fusion protein that is embedded in the plasma membrane and allows quantifiable analysis of the PMDS.</p>

                                <p class="text-content-new">We designed and tested two different transcription units (TU) containing a CMV promoter (BBa_K747096), a signal peptide with cleavage site (BM-40, BBa_K2170214), GFP (BBa_K648013), a TMD, PhoCl, mCherry, a NLS and a poly(A) tail (BBa_K1150012) with the aim of finding the most reliable TU that will produce a transmembrane protein embedded in the plasma membrane.The difference between the two TU’s is the TMD: In construct 1.0 the TMD is taken from TLR4 (TMTLR4, shown to work in Nagaraj et al 2011 and Qudrat & Truong 2016) and in construct 1.1 the TMD is taken from EGFR (TMEGFR, shown to work by the iGEM team Munich 2016).</p>

                                <p class="text-content-new">We used the SignalP 4.1 Server [Link] to predict signal peptides and cleavage sites, ProtScale Hydrophobicity Plot (Hphob. / Kyte & Doolittle) [Link] for analysis of a TMD in relation to the entire predicted structure and Transmembrane Analyzer [TMHMM] to predict the precise location of transmembrane domains inside our design.</p>

                                <p class="text-content-new"><b>Results:</b> Construct 1.0 and 1.1 both show the existence of a signal peptide with predictable cleavage site and a hydrophobic peak where the TMD is implicated. However, the TMTLR4 failed to be predicted as a reliable transmembrane domain in our construct.
                                    We, therefore, decided to assemble construct 1.1 and thereby further characterise TMEGFR which has previously submitted by iGEM Munich 2016 as a BioBrick.</p>

                                <p class="text-content-new"><b>Discussion:</b> The registry entry of TMEGFR says: “The EGFR stop transfer signal in the 3' region of the TMD increases the membrane stability. The part is ideal for combining internal and external membrane domains to one fusion protein. The glycine linkers in the 5' and 3' region of the part lead to a better flexibility and a better performance of the fused parts.The part is flanked by RFC25 pre- and suffix for further use in protein infusion.”</p>

                                <p class="text-content-new">Nagaraj et al. 2011 and Qudrat & Truong 2016 only give the amino acids of the TM­TLR4 but do not mention a ‘stop-transfer signal’ or the flanking sequences of their construct in general (not in the paper nor in the SI). However, the flanking sequence of a TMD (which is a stop-transfer anchor signal during co-translational import into the ER) is crucial in the establishment of a TMD and its orientation! his problem has been resolved by using the TMEGFR as described by iGEM Munich 2016.</p>


                            </div>
                        </div>

                        <!-- Experimental results-->
                        <div id="tissue-4" class="col-sm-11">
                            <div class="col-sm-11">
                                <p>results</p>
                            </div>
                        </div>


                        <!-- Bulb bits-->

                        <div id="bulb-1" class=" text-section col-sm-11">
                            <div class="col-sm-11">
                                <p class="text-content-new"> The reduction of greenhouse gas emissions (GHG) has become a common world-wide goal in response to the impacts of climate change and increasing global warming affecting numerous societies, ecosystems and wild-life. Implementation of LED lighting has been attempted to reduce GHG, however it is a very costly approach. We therefore developed the LIT Bulb, an efficient, sustainable, eco-friendly, long-term and cheap solution for public illumination that requires a minimal electricity/nutrient input (see figure 1). </p>


                                <img style="height: 40%; width:40%; margin:1% 0 1% 30% !important" src="http://2017.igem.org/wiki/images/2/2c/LIT-Bulb.png" class="image-content-hidden">

                                <p class="text-figure">Figure 1. Representation of our bacteria powered light bulb consisting of light sensitive bioluminescent <i>E. coli</i> cells co-cultured with surcrose secreting Cyanobacteria (<i>Synechococcus elongatus</i> PCC 7942)</p>

                                <p class="text-content-new"> The bulb will contain light-sensitive <i>E. coli</i> that detect levels of sunlight and regulate their bioluminescence, encoded by LuxCDABE, to only be active during night time. This is achieved through the incorporation of a photosensitive protein (EL222) and a blue-light repressible promoter (Pblrep) to regulate the bioluminescence of <i>E. coli</i>, in response to external sunlight levels. Additionally, engineered photosynthetic cyanobacteria, <i>Synechococcus elongatus</i> PCC 7942, will produce and secrete sucrose using heterologous sucrose transporters  to feed our recombinant <i>E. coli</i> for their long-term survival. </p>

                                <p class="text-content-new"> Through mathematical modelling we developed the optimised dimensions of the light bulb for maximum bioluminescence (160 watts). Finally, we did a cost analysis for the production of our LIT Bulb and developed a financial plan for its implementation into the current framework of street lighting in London, considering the current EU policy. </p>
                            </div>
                        </div>

                        <div id="bulb-2" class="col-sm-11">

                            <p class="text-content-new"> All current electricity generating methods have a carbon footprint. Almost half of our carbon footprint is due to electricity and 17% is due to lighting alone. In the UK alone (2005), street lighting was using approximately 3.14 TWH of electricity giving rise to CO2 emissions of 1.32 megatons. </p>

                            <p class="text-content-new"> - A single incandenscent light bulb produces 0.185 tonnes of  CO2 emissions yearly </p>
                            <p class="text-content-new"> - A single LED light bulb produces 0.185 tonnes of  CO2 emissions yearly </p>
                            <p class="text-content-new"> - Our LIT Bulb will produce only 0.002 tonnes of  CO2 emissions yearly </p>

                            <p class="text-content-new"> Into the future... </p>

                            <img style="height: 60%; width:60%; margin:1% 0 1% 20% !important" src="http://2017.igem.org/wiki/images/0/0e/London-lights.jpg">

                            <p class="text-content-new"> 1807: London’s Pall Mall becomes the first street in the world to be illuminated by gaslight. </p>
                            <p class="text-content-new"> 2057: The London Eye uses bioluminescence to impress tourists during night time. </p>


                        </div>

                        <div id="bulb-3" class="col-sm-11">

                            <p class="text-content-new"> The bulb will contain light-sensitive <i>E. coli</i> that detect levels of sunlight and regulate their bioluminescence (encoded by LuxCDABE gene cluster) to only be active during night time. LuxCDABE encodes the production of the proteins required for bacterial luminescence including luciferase and its substrates from Vibrio fischeri. Naturally, Vibrio fischeri use additional regulatory proteins (encoded by LuxR and LuxI upstream) whose external concentration increases as a function of increasing cell-population density. Bacteria detect the accumulation of a minimal threshold stimulatory concentration of these autoinducers and regulate the expression of LuxCDABE. Consequently, we removed these regulatory proteins to ensure that cell concentration dependent luminescence through quorum sensing is surpassed and our cultures can fluoresce independently. By using our  blue light repressible promoter (PBLrep), we will ensure that our bulb is only active in the dark and excess buildup of substrates and luciferase, potentially toxic, will be prevented. </p>

                            <p class="text-content-new"> Blue light repression is achieved through the incorporation of a photosensitive protein (EL222) and a blue-light repressible promoter (Pblrep). PBLrep consists of the 18bp DNA binding region of EL222, a natural photosensitive DNA-binding protein from the marine bacterium <i>Erythrobacter litoralis</i> HTCC2594, positioned between the -35 and -10 regions of the RNAP binding site. For blue light repression, cells must also express EL222 (BBa_K2332004). In the dark, EL222 is inactive as its N-terminal LOV domain represses its DNA-binding C-terminal HTH domain. In the daytime, exposure to blue light (465nm) results in the LOV-HTH interaction to be released, allowing it to dimerize and bind its binding region, causing steric hindrance to RNAP binding, ultimately repressing transcription. Therefore, only at nighttime the transcription of LuxCDABE will occur.
                            </p>

                            <img style="height: 60%; width:60%; margin:1% 0 1% 20% !important" src="http://2017.igem.org/wiki/images/b/be/Pblrep-LuxCDABE.png" class="image-content-hidden">
                            <p class="text-figure">Figure 1: Blue light repression system. Under blue light, the EL222 DNA binding protein dimerises and binds its binding region within the designed Pblind promoter, positioned between the -35 and -10 regions of the RNAP binding site, causing steric hindrance to RNAP binding, ultimately repressing transcription of LuxCDABE. Figure adapted from Jayaraman P. et al. (2016).</p>

                            <p class="text-figure">Planned experiment: Cells would have been transformed with PBLrep-LuxCDABE and  EL222 (constitutively expressing EL222 protein) plasmids. Cells would have been either kept in the dark or exposed to blue light (465 nm) for 6 hours and bioluminescence levels would have been measured by aliquoting samples into a 96-well plate every 6 hours using FLUOstar plate reader.
                            </p>

                            <p class="text-content-new"> Additionally, engineered photosynthetic cyanobacteria, <i>Synechococcus elongatus</i> PCC 7942, will produce and secrete sucrose using heterologous sucrose transporters  to feed our recombinant <i>E. coli</i> for their long-term survival. </p>

                            <img style="height: 65%; width:65%; margin:0 0 0 14% !important" src="http://2017.igem.org/wiki/images/archive/f/f8/20171027104659%21Ducat-cyano.png" class="image-content-hidden">

                            <p class="text-figure"> Figure 2. Engineered cyanobacteria co-cultured with <i>E. coli</i>. a) Depicts the mechanism of sucrose secretion through the expression of CscB sucrose transporter by <i>Synechococcus elongatus</i> 7942 (<i>S. elongatus</i>). b) Shows the optical density of the co-culturing of <i>E. coli</i>  and sucrose secreting <i>S. elongatus</i>. Optical density of the entire culture is shown with black points. Adapted from Ducat et al. 2017.</p>

                            <p class="text-content-new"> Sucrose secretion is tunable, stable, enhances photosynthetic efficiency of Cyanobacteria and as shown in figure 2, it has been demonstrated that co-cultures of <i>E. coli</i>  and sucrose secreting <i>S. elongatus</i> are stable over long periods of time when grown in constant light and CO2 as the only carbon source in the media. Sucrose-secreting <i>S. elongatus</i> can also be immobilized in Alginate beads suspended in our LIT Bulb to increase their total productivity on a per cell basis. </p>

                        </div>

                        <div id="bulb-4" class="col-sm-11">


                            <p class="text-content-new">Through mathematical modelling, the optimal dimensions and structure of the bulb were determined (click here for modelling details). The circular structure of the LIT bulb maximizes the exposure of Cyanobacteria to sunlight. A pump was introduced to ensure the cells remain suspended and the cell culture is homogeneously distributed throughout the LIT bulb. A three-way manual valve was introduced to allow for the easy replacement of media once every 12 months. A filter will be added to the valve to allow media to be replaced without removing the cell culture.</p>

                            <img style="height: 60%; width:60%; margin:0 0 1% 20% !important" src="http://2017.igem.org/wiki/images/3/33/Torus-bulb.png" class="image-content-hidden">

                            <p class="text-figure">Figure 1. Optimized design of the LIT Bulb for maximum bioluminescence. Design consists of a torus shape made out of plexiglass (4cm diameter by 95cm length). It contains a 3 way valve,  a small pump (1.8w) and a filter (0.015cm pore size) within the valve for biocontainment when replacing the media once a year.</p>


                            <p class="text-content-new">Biosafety and microbial containment considerations are of particular importance in the manufacturing design process of the LIT Bulb, as our product intends to coexist with the wider population as a source of public illumination.</p>

                            <p class="text-content-new">The bulb has been designed to ensure biocontainment and prevent the interaction with our engineered microorganisms or their release into the environment.</p>

                            <p class="text-content-new">The energy strain for the production of bioluminescence by our engineered <i>E. coli</i> cells is so large that they would be outcompeted quickly by other microorganisms if they were to be released in the environment.</p>

                            <p class="text-content-new">Cyanobacteria will be engineered to secrete the glucose product from photosynthesis, thus they would be unable to sustain their growth when competing against other microbes in the environment if they were to be released.</p>


                        </div>


                    </div>




                </div>

                <a href="#" id="myBtn" title="Go to top">Back to Top</a>

            </div>
        </div>

    </div>

</body>

</html>