<!DOCTYPE html>
<html lang="en">
<head>
    <script src="http://www.google.com/jsapi" type="text/javascript"></script>

    <script type="text/javascript">google.load("jquery", "1.3.2");</script>

    <meta charset="UTF-8">
    <title>AMethods</title>
    <link rel="stylesheet" type="text/css" href="../../css/template.css">

    <!-- BOOTSTRAP -->
    <!-- Latest compiled and minified CSS -->
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">

    <!-- jQuery library -->
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>

    <!-- Latest compiled JavaScript -->
    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>

    <script>

        $(document).ready(function(){
            //Take your div into one js variable
            var s1 = $("#step1");
            var s2 = $("#step2");
            var s3 = $("#step3");

            //Take the current position (vertical position from top) of your div in the variable
            var pos = s1.position();
            var pos1 = s2.position();
            var pos2 = s3.position();

            //Now when scroll event trigger do following
            $(window).scroll(function () {

                scrollFunction()

                var windowpos = $(window).scrollTop();
                //console.log(windowpos);

                //Now if you scroll more than 100 pixels vertically change the class to AfterScroll
                if (windowpos >= (pos.top + 100)) {
                    // div.addClass("AfterScroll");
                    s2.removeClass("active-link");
                    s1.addClass("active-link");
                }

                if (windowpos >= (pos1.top + 750)) {
                    // div1.addClass("AfterScroll");
                    s1.removeClass("active-link");
                    s3.removeClass("active-link");
                    s2.addClass("active-link");

                }

                if (windowpos >= (pos2.top + 1100)) {
                    //div2.addClass("AfterScroll");
                    s2.removeClass("active-link");
                    s3.addClass("active-link");
                }

            });

            function scrollFunction() {
                if (document.body.scrollTop > 50 || document.documentElement.scrollTop > 50) {
                    document.getElementById("myBtn").style.display = "block";
                } else {
                    document.getElementById("myBtn").style.display = "none";
                }
            }

        });
    </script>

    <!-- parameter tables drop dpwn-->
    <script>
        $(document).ready(function(){
            $("#show1").click(function(){
                $("#hide1").slideToggle("slow");
            });
            $("#show2").click(function(){
                $("#hide2").slideToggle("slow");
            });
        });

    </script>
</head>

<style>
    .image-content-here {
        height:400px;
        width:800px;
    }
</style>

<body data-spy="scroll" data-target="#myScrollspy" data-offset="15">
<!-- The top navbar -->
<div class="container-fluid">
    <div id="top" class="row">

        <ul class="navy">
            <li id="drop1" class="navy-link"><a href="#">PROJECT</a>
                <ul class="navy-hidden">
                    <li><a href="#" class="dropped">Description</a></li>
                    <li><a href="#" class="dropped">Experiments</a></li>
                    <li><a href="#" class="dropped">Results</a></li>
                    <li><a href="#" class="dropped">Parts</a></li>
                    <li><a href="#" class="dropped">Gold Criteria</a></li>
                    <li><a href="#" class="dropped">Interlab</a></li>

                </ul>
            </li>

            <li id="drop4" class="navy-link"><a href="#">NOTE BOOK</a>
                <ul class="navy-hidden">
                    <li><a href="#" class="dropped">Protocols</a></li>
                    <li><a href="#" class="dropped">Safety</a></li>
                    <li><a href="#" class="dropped">Stuff</a></li>
                </ul>
            </li>
            </li>

            <li id="drop3" class="navy-link"><a href="#">ENTREPRENEURSHIP</a>
                <ul class="navy-hidden">
                    <li><a href="#" class="dropped">Overview</a></li>
                    <li><a href="#" class="dropped">Product</a></li>
                    <li><a href="#" class="dropped">IP</a></li>
                    <li><a href="#" class="dropped">Business Environment</a></li>
                    <li><a href="#" class="dropped">Marketing</a></li>
                    <li><a href="#" class="dropped">Management</a></li>
                    <li><a href="#" class="dropped">Economic Appraisal</a></li>
                </ul>
            </li>

            <li id="drop5" class="navy-link"><a href="#">MODELLING</a>
                <ul class="navy-hidden">
                    <li><a href="#" class="dropped">Overview</a></li>
                    <li><a href="#" class="dropped">Barchitecture</a></li>
                    <li><a href="#" class="dropped">Game of LIT</a></li>
                    <li><a href="#" class="dropped">Mammalian</a></li>
                    <li><a href="#" class="dropped">Light Bulb</a></li>
                    <li><a href="#" class="dropped">Glossary</a></li>
                </ul>
            </li>

            <li id="drop" class="navy-link"><a href="#">HUMAN PRACTICES</a>
                <ul class="navy-hidden">
                    <li><a href="#" class="dropped">Overview</a></li>
                    <li><a href="#" class="dropped">Public Engagement</a></li>
                    <li><a href="#" class="dropped">Integrated Practices</a></li>
                </ul>
            </li>

            <li id="drop2" class="navy-link"><a href="#about">JUDGING CRITERIA</a>
                <ul class="navy-hidden">
                    <li><a href="#" class="dropped">Bronze</a></li>
                    <li><a href="#" class="dropped">Silver</a></li>
                    <li><a href="#" class="dropped">Gold</a></li>
                </ul>
            </li>

            <li id="drop6" class="navy-link"><a href="#about">PEOPLE</a>
                <ul class="navy-hidden">
                    <li><a href="#" class="dropped">Team</a></li>
                    <li><a href="#" class="dropped">Collaborations</a></li>
                    <li><a href="#" class="dropped">Attributions</a></li>
                </ul></li>

        </ul>
    </div>

</div>

<div class="container-fluid">
    <div class="row">

        <!-- The side navbar -->
        <nav class="col-sm-2">
            <ul id="side-index" class="nav nav-stacked" data-spy="affix" data-offset-top="205">
                <li id="step1" class="navy-link side-link"><a href="#section1">Overview</a></li>
                <li id="step2" class="navy-link side-link"><a href="#section2">Sensitivity Analysis</a></li>
                <li id="step3" class="navy-link side-link"><a href="#section3">Parameter Sampling</a></li>
                <li id="step4" class="navy-link side-link"><a href="#section4">Bibliography</a></li>
            </ul>
        </nav>

        <!-- Content -->
        <div style="float: right !important" class="col-sm-10">

            <div style="margin-top: 20px" class="text-section">

                <h1 class="title">Cracking the Model</h1>
                <h2 class="subtitle">Methods of parameter optimisation</h2>

                <p class="text-content"> Mathematical Modelling describes complex biological systems with mathematical
                    equations and concepts. Setting up the initial conditions in the model is not that straight forward.
                    If no experimental data is available, reasonable assumptions based on literature, logic and
                    quantitative analysis need to be made. We believe that the initial assumptions made are crucial
                    to crack the model. Once, established the parameters need to constantly be optimised with experimental
                    data and/or literature data in order to fit reality to its maximum. Parameter optimisation adds all
                    the weight to a model's credibility, this is why we decided to approach this in 2 ways:
                    Sensitivity Analysis and Parameter Sampling.
                    <br>
                    <h2 class="linktitle">Scroll down to find out more! </p>
            </div>

            <div class="text-section">
                <h2 class="title">Sensitivity Analysis</h2>

                <p class="text-content">Sensitivity Analysis is a conventionally used method for parameter optimisation.
                    A range of independent variables is tested in this technique and their impact on the dependent variable is evaluated.
                    In our case, we  want to optimise the impact specific values will have on the rate limiting step of the cellular mechanism,
                    So we are evaluating a range of values and analysing which ones optimise the proces. We applied this method of parameter optimisation to
                    the LEGIT and the LAME models.</p>

                <h2 class="subtitle">Method</h2>
                    <ol class="text-content" style="width: 100%">
                        <li style="margin-left: 20px">Identify the dependent variable to be optimised (the rate limiting step in our process)</li>
                        <li style="margin-left: 20px">Identify the independent variables that have an impact on this step</li>
                        <li style="margin-left: 20px">From literature search/ experimental data, obtain a range of values to be tested for each independent variable</li>
                        <li style="margin-left: 20px">Run the model for the range of values tested and identify the optimal condition for the rate limiting step</li>
                        <li style="margin-left: 20px">Re-run the initial model to evaluate the impact of the optimised parameters on the overall process, i.e. to determine if the optimised step is prevailed the rate limiting step in the overall cellular mechanism</li>
                        <li style="margin-left: 20px">Determine the areas in which research could be performed to further optimise the parameters (Wet Lab contribution)</li>
                    </ol>

                <div id="show1" class="text-section">
                    <p><i class="down" ></i> </p>
                        <h2 class="tabletitle">Sensitivity Analysis method applied to LEGIT model</h2>
                    </div>
                    <div id="hide1">
                        <p class="text-content" style="margin-left: 20px; padding:5px">
                            Given that transport of intimin to the surface of the E.coli cells is the rate
                            limiting step in the overall process, we are rewriting this ODE in order to simplify
                            the Sensitivity Analysis on this step. </p>
                        <img src="../../assets/modellingMethods/sensitivityAnalysisEqu.png" style="width: 700px; height:700px; margin-top:10px">
                    </div>
                </div>


            <div class="text-section">
                <h2 class="title">Parameter Sampling</h2>
                <p class="text-content">Parameter sampling, is another technique used to tackle the issue of uncertain and incomplete knowledge of parameter values.
                    It is a method of parameter optimisation, which relies on the random sampling of parameter values within a specified range.
                    A probability density function (PDF) determines the distribution of the continuous random variables evaluated in the function.
                    Parameter sampling considers biological noise, making it a more realistic representation of the behaviour of a biological system.</p>

                <h2 class="subtitle">Method</h2>
                    <ol class="text-content" style="width: 100%">
                        <li style="margin-left: 20px">Identify the dependent variable to be optimised (the rate limiting step in our process)</li>
                        <li style="margin-left: 20px">Identify the independent variables that have an impact on the step we are trying to optimise</li>
                        <li style="margin-left: 20px">Collect raw data from literature</li>
                        <li style="margin-left: 20px">Process raw data by assigning weights to the values depending on the similarity to the to-be-optimised parameter</li>
                        <li style="margin-left: 20px">Calculate the weighted mean for all the optimised parameters</li>
                        <li style="margin-left: 20px">Generate probability densitiy functions (pdf);input the sample size, standard deviation and weighted mean</li>
                    </ol>

                <h2 class="subtitle">LEGIT model</h2>
                <p class="text-content">Assessing the impact of randomly generated discrete variables on the rate of
                    transport of intimin. For this the values of K<sub>m</sub> and V<sub>max</sub> are evaluated. </p>

                        <!-- add the tree diagram-->

                    <ul class="text-content" style="width: 100%"> Conditions for intimin
                        <li style="margin-left: 20px">Size: 94 kDa </li>
                        <li style="margin-left: 20px">Optimal pH conditions: 7 </li>
                        <li style="margin-left: 20px">Native organism: E.coli </li>
                    </ul>

                <p class="text-content" style="padding-top: 0">Weighted mean of K<sub>m</sub>: 0.0147 uM</p>
                <p class="text-content" style="padding-top: 0">Weighted mean of V<sub>max</sub>: 300000 s<sup>-1</sup>(1 s.f.) </p>

                <p class="linktitle"> Click here to check out the data collected to optimise the value of K<sub>m</sub> </p>
                <!-- ADD LINK for the table of values-->
                <p class="linktitle"> Click here to check out the data collected to optimise the value of V<sub>max</sub> </p>
                <!-- ADD LINK  for the table of values -->


                <h2 class="subtitle">LAME model</h2>
                <p class="text-content">Assessing the impact of randomly generated discrete variables on the rate of translation of GFP. For this the rate of nuclear export of the transcript and the rate of GFP degradation are evaluated.</p>

                <!-- add the tree diagram-->

                <ul class="text-content" style="width: 100%"> Conditions for the RFP protein
                    <li style="margin-left: 20px">Size: 27.9 kDa </li>
                    <li style="margin-left: 20px">Location of protein: Cytoplasm</li>
                    <li style="margin-left: 20px">Native organism: CHO cells </li>
                </ul>

                <p class="text-content" style="padding-top: 0">Weighted mean of rate of nuclear export: 1.43 x 10<sup>-4</sup> s<sup>-1</sup></p>
                <p class="text-content" style="padding-top: 0">Weighted mean of rate of degradation: 8.96 x 10<sup>-1</sup> s<sup>-1</sup></p>

                <p class="linktitle"> Click here to check out the data collected to optimise the  rate of nuclear export</p>
                <!-- ADD LINK for the table of values-->
                <p class="linktitle"> Click here to check out the data collected to optimise the rate of degradation</p>
                <!-- ADD LINK  for the table of values -->
            </div>


                <div class="text-section">

                        <h1 class="title">Selecting our Programming Languages </h1>
                        <p class="text-content"> This summer we decided to run our models on two different programming languages:
                            MATLAB and Python. Both are high level languages that allowed us to create robust simulations.
                            Upon starting iGEM, everyone on the modelling team wanted to learn a new programming language.
                            Python is an open source software and it is relatively easy to find resources online to help
                            troubleshoot our codes. We also decided to use MATLAB for our models, as we were already familiar
                            with this language from our engineering degrees. </p>
                    </div>



            <div class="text-section">

                <h6 class="title" style="color:rgba(204, 0, 53, 0.69)">Bibliography</h6>
                    <p class="subtitle">
                        <li>Thereza, C., Marina, I., Luciana.Cambricoli, d. and Olivia, C. (2004). Expression of green fluorescent protein (GFPuv) in Escherichia coli DH5-a, under different growth conditions. African Journal of Biotechnology, 3(1), pp.105-111.</li>
                        <li>Wlab.ethz.ch. (2017). Cell Surface Protein Atlas. [online] Available at: http://wlab.ethz.ch/cspa/#abstract [Accessed 4 Sep. 2017].</li>
                        <li>Sciencedirect.com. (2017). Membrane integration of E. coli model membrane proteins - ScienceDirect. [online] Available at: http://www.sciencedirect.com/science/article/pii/S0167488904000813 [Accessed 4 Sep. 2017].</li>
                        <li>Arun KH e. Green fluorescent proteins in receptor: an emerging tool for drug discovery. – PubMed –NCBI [Internet]. Ncbi.nlm.gov.2017 [cited 21 August 2017]. Available from: https://www.ncbi.nlm.nih.gov/pubmed/15596111</li>
                        <li>[internet] UVGermicidalTechnicalData. [cited 10 September 2017] http://www.pveducation.org/pvcdrom/effect-of-light-intensity (accessed 10 September 2017).</li>
                        <li>Jayaraman, P., Devarajan, K., Chua, T. K., Zhang, H., Gunawan, E. and Loo Poh, C.  Blue light-mediated transcriptional activation and repression of gene expression in bacteria. NCBI. 2016;44(14):6994–7005.</li>
                        <li>Philips, Ron. "Cell Biology By The Numbers." Book.bionumbers.org. N.p., 2017. Web. 10 Sept. 2017.</li>
                        <li>Kiparissides, Alexandros et al. "Modelling The Delta1/Notch1 Pathway: In Search Of The Mediator(S) Of Neural Stem Cell Differentiation." PLoS ONE 6.2 (2011): e14668. Web.</li>
                        <li>Carbonell-Ballestero M, Duran-Nebreda S, Montanez R, Sole R, Macia J, Rodriguez-Caso C. A bottom-up characterization of transfer functions for synthetic biology designs: lessons from enzymology. 2017.</li>
                        <li>Km & Vmax [Internet]. Mofetsrv.mofet.macam98.ac.il. 2017 [cited 21 September 2017]. Available from: http://mofetsrv.mofet.macam98.ac.il/~rafid/BIOLOGY/biochem/infopages/aminoacids/www-biol.paisley.ac.uk/courses/stfunmac/glossary/Km.html</li>
                        <li>Ramakrishnan P, Maclean M, MacGregor S, Anderson J, Grant M. Cytotoxic responses to 405nm light exposure in mammalian and bacterial cells: Involvement of reactive oxygen species. 2017.</li>
                        <li>Zitieren eine Webseite - Cite This For Me [Internet]. Karger.com. 2017 [cited 28 September 2017]. Available from: https://www.karger.com/Article/Pdf/131439</li>
                        <li>HeLa cell volume - Human Homo sapiens - BNID 103725 [Internet]. Bionumbers.hms.harvard.edu. 2017 [cited 28 September 2017]. Available from: http://bionumbers.hms.harvard.edu/bionumber.aspx?id=103725&ver=3</li>
                        <li>Baumann N, Vidugiriene J, Machamer C, Menon A. Cell Surface Display and Intracellular Trafficking of Free Glycosylphosphatidylinositols in Mammalian Cells. 2017.</li>
                        <li>Cortes L, Vainauskas S, Dai N, McClung C, Shah M, Benner J et al. Proteomic identification of mammalian cell surface derived glycosylphosphatidylinositol-anchored proteins through selective glycan enrichment. 2017.</li>
                        <li>Key Numbers for Cell Biologists [Internet]. Bionumbers.org; 2017 [cited 28 September 2017]. Available from: http://bionumbers.hms.harvard.edu/Includes/KeyNumbersLinks.pdf</li>
                        <li>Ramakrishnan P, Maclean M, MacGregor S, Anderson J, Grant M. Cytotoxic responses to 405nm light exposure in mammalian and bacterial cells: Involvement of reactive oxygen species. 2017.</li>
                    </p>
            </div>


        <a href="#" id="myBtn" title="Go to top">Back to Top</a>

    </div>
</div>

</body>

</html>